# Cross-branch diff. Build 26100.7653 is from ge_release_svc_betaflt_prod1, 26100.7674 is from ge_prerelease_im.

@@ -24,7 +24,6 @@
   1542029624: 54305864
   1568623929: 59703561
   1595708730: 58300873
-  1623501113: 58093318
   1650239802: 58116550
   1658105146: 58190790
   1691920698: 58091719
@@ -35,12 +34,10 @@
   1831114041: 58267397
   183181624: 60089436
   1855570234: 53278917
-  1877249336: 58697541
   1892530489: 53636354
   191139130: 58689500
   1933484349: 49582338
   1952911675: 60311427
-  2022925624: 59079232
   2029308216: 57566016
   2045735225: 58729728
   2098154810: 59710913
@@ -49,12 +46,10 @@
   2112313656: 59385665
   2119215417: 58116609
   2136437049: 58755841
-  2193672504: 57601662
   2203281722: 59149054
   2217258297: 58151999
   2227486009: 57896767
   231696698: 59125213
-  2383447353: 58579517
   2397524283: 59303869
   23H2_Enablement: 45796396
   2410108218: 59357181
@@ -64,7 +59,6 @@
   2473284922: 59344122
   2488582459: 57428923
   2509806905: 58406971
-  2575004987: 58785976
   2601742649: 58759736
   26370069: 26370069
   2673920312: 57575033
@@ -79,7 +73,6 @@
   2748606776: 54963062
   2835917115: 58072244
   2905561403: 59381173
-  3014704440: 57913970
   3137401147: 56663728
   3137640762: 58367984
   3159936315: 57624753
@@ -110,7 +103,6 @@
   35409324: 35409324
   36085911: 36085911
   36435131: 36435131
-  3666920762: 57539304
   3668653368: 59896936
   3695655224: 59920233
   3743102264: 59873385
@@ -119,12 +111,10 @@
   3864166716: 50121319
   3877411128: 58366311
   38781006: 38781006
-  3879239994: 58495207
   38841075: 38841075
   38931291: 38931291
   38f13e0fba0656bd8248c0fdddc56de1: 32145346
   3909565754: 59973604
-  3929572667: 58577060
   3932647737: 58771236
   3942159674: 58319076
   3955695931: 60075940
@@ -152,7 +142,6 @@
   43698028: 43698028
   43699481: 43699481
   44077960: 44077960
-  443063611: 58442904
   44519444: 44519444
   44533041: 44533041
   44571814: 44571814
@@ -187,7 +176,6 @@
   47251916: 47251916
   47692516: 47692516
   47826483: 47826483
-  47944061: 47944061
   47988461: 47988461
   48005687: 48005687
   48101911: 48101911
@@ -229,9 +217,7 @@
   50566657: 50566657
   50582925: 50582925
   50597829: 50597829
-  50908904: 50908904
   510094651: 59230105
-  51192705: 51192705
   51215370: 51215370
   51451849: 51451849
   51473953: 51473953
@@ -244,10 +230,8 @@
   51956543: 51956543
   519795000: 58777689
   52049001: 52049001
-  52063867: 52063867
   52470297: 52470297
   52754209: 52754209
-  52845862: 52845862
   52940282: 52940282
   52986630: 52986630
   52988742: 52988742
@@ -325,9 +309,9 @@
   55370801: 55370801
   55374219: 55374219
   55383831: 55383831
+  55385725: 55385725
   55433877: 55433877
   55463899: 55463899
-  55482414: 55482414
   55495322: 55495322
   55524821: 55524821
   55535445: 55535445
@@ -336,7 +320,6 @@
   55659301: 55659301
   55666696: 55666696
   55679201: 55679201
-  55681739: 55681739
   55700834: 55700834
   55700942: 55700942
   55709058: 55709058
@@ -361,6 +344,7 @@
   55888780: 55888780
   55911954: 55911954
   55934612: 55934612
+  55986800: 55986800
   55991761: 55991761
   56038648: 56038648
   56039577: 56039577
@@ -382,7 +366,6 @@
   56224408: 56224408
   56231474: 56231474
   56245971: 56245971
-  56246287: 56246287
   56256542: 56256542
   56257248: 56257248
   56268098: 56268098
@@ -479,7 +462,6 @@
   56998697: 56998697
   57003568: 57003568
   57003800: 57003800
-  57004311: 57004311
   57004514: 57004514
   57006869: 57006869
   57012083: 57012083
@@ -691,6 +673,7 @@
   58475882: 58475882
   58484184: 58484184
   58500308: 58500308
+  58537189: 58537189
   58544091: 58544091
   58546274: 58546274
   58548552: 58548552
@@ -733,6 +716,7 @@
   58937533: 58937533
   58938944: 58938944
   58939878: 58939878
+  58939894: 58939894
   58992016: 58992016
   58994121: 58994121
   59011689: 59011689
@@ -833,6 +817,7 @@
   59961277: 59961277
   59964684: 59964684
   59977655: 59977655
+  59978822: 59978822
   5cdff46ad785473187b04fe5b23e1709: 44498562
   60007725: 60007725
   60038143: 60038143
@@ -848,6 +833,7 @@
   60181467: 60181467
   60191229: 60191229
   60204084: 60204084
+  60207067: 60207067
   60215970: 60215970
   60237021: 60237021
   60237387: 60237387
@@ -857,6 +843,7 @@
   60295108: 60295108
   60295117: 60295117
   60311650: 60311650
+  60341450: 60341450
   60353309: 60353309
   60360690: 60360690
   60360782: 60360782
@@ -869,10 +856,13 @@
   60473571: 60473571
   60475022: 60475022
   60475287: 60475287
+  60485750: 60485750
   60487745: 60487745
   60513883: 60513883
   60541990: 60541990
+  60551673: 60551673
   606399802: 57104343
+  60672206: 60672206
   607cd3c1c69c4b20b2a4cd4c1bb48b3d: 52552402
   612857145: 58818327
   612938042: 57703639
@@ -959,10 +949,9 @@
   AddPreloadCheckToOtherEnduserOobePages: 58706937
   AddRulesEngineTipTest: 53517471
   AddSourceWindowFromViewNavigationRequested: 9205753
-  AddStrongRefsToSearchAppearanceAsyncCalls: 48194277
   AddSwitchAsyncContextToPositionerActivations: 9582236
-  AdditionalClocksSettings_BugFix: 58682398
   AdminFlowRejuv_AddDomainUserAndEditUser: 59359094
+  AdminFlowRejuv_CreateVhdPrompt: 59783595
   AdminFlowRejuv_EnterProductKeyPage: 42733866
   AdminFlowRejuv_PhoneActivationPage: 42739800
   AdminFlowRejuv_RetailDemoConfirmPage: 44357190
@@ -974,6 +963,8 @@
   AgentActivationRuntimeAPI_AboveLock: 20829994
   AgentSessionActivationSignalsTest_60236308: 60236308
   AgentSessionApiActivationTest_59098673: 59098673
+  AgileActivationFactories: 60444501
+  Aidd_InventorySvc: 58976982
   AllKeysAndBootMgrByWinCS: 58304633
   Allow3POROMRollout: 54350110
   Allow3PUEFICARollout: 55061616
@@ -1003,7 +994,6 @@
   AppDefaultHashRotationUpdateHashes: 27623730
   AppDefaultHashRotationV1ToV2: 58679359
   AppDefaultSkipVerifyBeforeMigration: 59144945
-  AppDefaultVolatileKeyError: 48456790
   AppDefaultsOptimizeStoreDetection: 59089691
   AppDefaultsRemoveUninstalledStoreItem: 58072730
   AppExtensions_CacheFiltering_AllowDockedSystemAppPackages: 59432242
@@ -1053,7 +1043,6 @@
   AudioSetFormatError: 60127125
   AugmentedEntryPoint: 36226054
   AugmentedEntryPointBadging: 36718398
-  AutoDismissLockTelemetry: 48697684
   AutoEscalationSupport: 59872159
   AutoGameMode: 11015710
   AutoGameModeProfile: 11650261
@@ -1075,17 +1064,18 @@
   AvailableLanguagesToolTip: 52294697
   AverageAggregation: 38513938
   AvoidLoadingLegacyOfficeSpeller: 42911515
-  B2504: 0
   B2508: 0
   BELFSCF: 56279229
   BFGAA: 57345194
   BFIND: 57927587
   BLHFWF: 60367327
+  BPAMF: 60664762
   BPT_CD_BugFixes_2505: 56203695
   BPT_CD_BugFixes_2507: 56844971
   BRC_TelemetryImprovement: 52446107
   BTCBF: 50617828
   BTPanUI: 41540195
+  BUTFF: 59279894
   BV: 55642965
   BVABF: 59077552
   BVFI: 59107546
@@ -1107,7 +1097,6 @@
   BalloonCancellationFix: 57543648
   BaseImageUsingServicingStack: 41024749
   BaselineMetadataAcquisition: 43396183
-  BatchFileCodeIntegrity: 35140618
   BetterToastTip: 57616363
   BfeSubnets_BugFixes_25B: 51772718
   Bfe_Timer_BugFixes_25B: 52758005
@@ -1141,20 +1130,17 @@
   Brain_v14SearchesForP3_Server: 58614663
   BreadCrumbBarMemoryLeak: 56788622
   BreadcrumbBarLeafNodeToolTip: 56390325
-  BreadcrumbDragAndDrop: 47664723
   BrokerCleanupSBCandDowntime: 58433260
   BubblesStackOverflowFix: 53846605
   BufferCommands: 7471763
   BugFix39591904_CloudClipboardSetting: 39591904
-  BugFix49347806_KeyboardFocusVisual: 49347806
   BugFix58139638: 58139638
   BugFix59071307: 59071307
-  BugFixCHSConversionListStyle: 33343659
+  BugFix60248196: 60248196
   BugFixJsonHelperCannotBeUsedInMTA: 59897666
   BugFixOobeTouchKeyboardLaunch: 49046212
   BugFixTCPOnPublicNetwork: 55473919
   BugFixWAMException: 56619032
-  BugFix_49347877_WebviewAndProgressRingPadding: 49347877
   BugFix_55602774: 55602774
   BugFix_57632791: 57632791
   BugFix_57694803: 57694803
@@ -1165,7 +1151,6 @@
   BugFix_RDX_NightLightStuckState: 42669370
   BugFix_RDX_SetAccountPictureTask: 40102193
   BugFix_SessEnvOobRead: 56035949
-  BugFix_SessEnvThreadHandleManagement: 54535558
   BugFixes2D: 60295199
   BugFixesFor10D: 58471571
   BugFixesQ3: 59125405
@@ -1203,7 +1188,6 @@
   Bugfix_WiFiGpUxFipsMode: 49800673
   BusBF10: 0
   Bypass3rdPartyKspsInRenew: 58379160
-  BypassSCCDRequirementForInboxPackages: 44169777
   CAMBugBundle_Se: 53496814
   CAMStorageDatabase: 54365364
   CAMStorageDatabase_Wave1: 55578545
@@ -1233,6 +1217,7 @@
   CDVACSI: 55315192
   CDVADMBF: 59807637
   CDVADOOBE: 57457659
+  CDVAEDLS: 60621895
   CDVAEJAJPLS: 59037714
   CDVAELS: 58136929
   CDVAFAG: 58560373
@@ -1245,13 +1230,17 @@
   CDVAIFA: 58730079
   CDVALLASNS: 57428401
   CDVALSCF: 58555008
+  CDVANAKFF: 59730436
   CDVANDE: 59668702
   CDVANFMW: 57508752
   CDVANFRE: 58680439
   CDVANSWF: 57322519
+  CDVANVAWUP: 59243407
   CDVAPSDD: 59459092
   CDVARTPCF: 56875575
+  CDVASFF: 60563076
   CDVASNA: 59669219
+  CDVASNTFPF: 60237108
   CDVASSL: 57696182
   CDVATC: 57828830
   CDVATCLS: 59887882
@@ -1262,6 +1251,7 @@
   CDVTEC: 59825425
   CDVTPD: 58970402
   CDVTTC: 57864869
+  CD_LCCIFSM: 60605216
   CD_LCDM: 57836660
   CD_LCLTT: 60240760
   CD_LCNIL: 55604124
@@ -1289,7 +1279,6 @@
   CNGRefactor: 54031803
   CShellOnWindowManagerIDK: 22855141
   CTTAInAdminlessLocalizationFix: 59713996
-  CTTATextboxPlaceHolderStringFix: 57504037
   CUDIEX: 47796414
   CUICOORD: 28330126
   CXHAPIBaseTrustWinRTRestriction: 57043193
@@ -1313,12 +1302,12 @@
   Census_DeviceName_GroupPolicy: 57656335
   Census_InternalNumberOfDisplays: 57716718
   Census_InventorySvcStartType: 56694530
-  Census_IsWindows365Device: 56521690
   Census_IsWindows365Device_CTAC: 58706147
   Census_Remove_EnableLogging: 51275296
   Census_WMI_TPM: 55676715
   CentennialInUIJob: 58607834
   CentennialLaunchUriForResults: 51724290
+  CertPinningCheck: 60245809
   ChangeSettingsAppMinimumWidth: 54915060
   CharMapAccessibilityFix: 46115212
   CharMapAccessibilityFix_V2: 48849268
@@ -1331,7 +1320,6 @@
   CheckWallPaperAndThemePolicy: 46592582
   ChildBitFlipToZeroSync: 58303482
   ChildHwndForIslandInputSite: 49204999
-  CiliumOnWindows: 49427273
   CimBoot: 31577186
   CimfsStackEventTracerFix: 57552994
   ClaimRawDiskRaceCondition_Fix: 55676103
@@ -1348,8 +1336,8 @@
   CloudDirectSubscribeNotification: 54288158
   CloudPCCommon_v3: 47542902
   CloudPCFastSwitchDiscovery: 40459297
-  CloudPopulatedTipMeasures: 40246772
   CloudRequestorReadDirect: 54902164
+  CmdAllowLockBatchFilesOptIn: 59769949
   ColdStartSuggestions: 60240051
   CollectTebPebForEmulatedProcesses: 57899610
   ColorBackupHandler: 34853364
@@ -1370,6 +1358,7 @@
   CommandsRecommendedCardsCraftFix: 59908638
   CommandsRecommendedCardsPerformanceImprovement: 60448530
   CommandsRecommendedCardsRemoveDisplayResolution: 60597149
+  CommandsRecommendedCardsRestrictionToCPPlusPC: 60570144
   CommandsRecommendedCardsUpdate: 58264966
   CommandsUxNotRefreshingCorefix: 56861570
   CommercialCloudAppBackup: 57382763
@@ -1407,9 +1396,6 @@
   ConfirmAdminStorageSettings: 57307354
   ConnectionQualityAPI: 46735773
   ConsoleLogonUXUseDisplayLangCodePage: 27866248
-  ContainerNetBugFixes2503: 55390665
-  ContainerNetBugFixes2506: 56092729
-  ContainerNetBugFixes2511: 57512403
   Containment_AHUpdateBeforeRebootEval_59110738: 59110738
   Containment_AddScopeExitForTelemetryActivity_56286335: 56286335
   Containment_AddedTelemetryForPastScheduling_41028971: 41028971
@@ -1697,7 +1683,6 @@
   CrossSessionBamos: 59070177
   CtacSessionContext: 32137669
   CurfewAllowanceCalculation: 33207570
-  CustomDefaultFrameSizing: 43769745
   CustomKeyboardProfiles: 35590764
   CustomServerError: 43044814
   CustomXamlUIDialog: 34037864
@@ -1715,14 +1700,13 @@
   DDAConnectorFallbackInCollab: 58976598
   DFN: 57345863
   DL44925748: 44925748
-  DL47854811: 47854811
   DL54475268: 54475268
-  DL55870273: 55870273
   DL57150615: 57150615
   DL58267129: 58267129
   DL58443624: 58443624
   DL58995572: 58995572
   DL59424588: 59424588
+  DL60280066: 60280066
   DLDPF: 58275923
   DLSDOCF: 54445996
   DLSDR: 52514267
@@ -1735,7 +1719,6 @@
   DSCN: 54920739
   DSID: 57808559
   DSSvc_RedirectionGuard: 39833203
-  DST: 57573012
   DTAI: 57470233
   DTBF_60535942: 60535942
   DWWC: 59302422
@@ -1795,7 +1778,7 @@
   DeleteMinuteZeroExperiences: 56347080
   DemandStartNetsetupInAI: 59203798
   DependencyTest: 50203200
-  DeprecateCoreEditViewInputScope: 41082674
+  DeprecateDeadCode2602: 60496648
   DeprecatePre11Styles: 60454545
   DesktopApps_OffByDefault: 57708093
   DesktopReadyNotif: 44768719
@@ -1859,15 +1842,16 @@
   DevDriveHyperlinkFix: 59783560
   DevInv_NormalizedPath_Overflow: 59707718
   DeviceBasedLicensing: 28179707
+  DeviceHealth: 60497802
   DeviceHotTrace: 58497816
   DeviceInstall_AdditionalCatalogAttributeLogging: 60235994
   DeviceInstall_CatalogAttributesLogging: 55755603
+  DeviceInstall_CheckDriverIntegrity: 58033796
   DeviceInstall_PreprodCatalogAttributeLogging: 57466833
   DeviceInstall_TimeoutReset: 58574132
   DeviceIntegrationPolicy: 44532601
   DeviceIntegrationPolicyWXH: 44740108
   DeviceNameRebootMove: 59353802
-  DeviceSetup_FixIssueInOobe: 56598111
   DialogTipFocus: 42235882
   DialogTipForeground: 39726108
   DigitsSubstitutionSetting: 20516266
@@ -1876,6 +1860,7 @@
   DisableAddAccountForFamilyNonOrg: 36660568
   DisableConsentReadFromCDS: 56259675
   DisableContextMenus: 36435038
+  DisableCustomCachingInIAttachementExecute2_59705529: 59705529
   DisableDesktopSpotlightOnFatalFailure: 43612613
   DisableDevHomeLaunchOnUserIntentSettingsPage: 60270958
   DisableDirectCompositionOfferReclaim: 56134111
@@ -1899,7 +1884,6 @@
   DisplayProvider: 36878944
   DisplaySearchSettingsForStandardUsers: 49757605
   DmaCommonSchema: 43970368
-  DmaSsoAADCompliance: 60413865
   DmaUtcUpdate: 43970355
   DoNotDisturbMigrationLogOnTask: 48543305
   DoNotPublishUserActivitiesToAfc: 51878560
@@ -1915,6 +1899,7 @@
   DriverSetup_CopyInfPresence: 56640376
   DriverSetup_ReofferFingerprinting: 49926024
   DriverSetup_WuPropertyLogging: 57083028
+  DriverStore_EnumerateBlockedDrivers: 57880228
   DriverUpdateOsVersionChange: 45034969
   DualEngine_AllButCurrentEntryDeletedDeserialization: 44344386
   DualEngine_AllowF12: 23071760
@@ -1931,7 +1916,6 @@
   DualEngine_SuppressUnneededZoomEvents: 29059948
   DualEngine_SynchronousZoom: 29078494
   DualEngine_UninitializeCouplingOnClose: 43897227
-  Dui70NullCrash: 60445073
   DuplicateTilesInAllAppsFix: 57204904
   DwmNotifZeroSizeNormalFix: 58050785
   DxDBHelper_AppStatus: 54016331
@@ -1946,9 +1930,9 @@
   EE_MPTF_Functionality: 49519257
   EFCIsWindowsUpdateManagedVariantFix: 59816929
   EID: 48504539
+  EKBAwareBranding: 60552733
   ENTERPRISE_MANAGER_CLOSEKEY: 58937320
   EPDA: 46122933
-  EPRF: 56673342
   EPUB_HostExtension: 15477137
   ESCFSRPF: 56786993
   ESIM_ALWAYS_PROCESS_ATR_NOTIFICATION: 53991482
@@ -1981,7 +1965,6 @@
   EmbeddedFetch: 12274531
   EmbeddedPrint: 13876261
   EmiChannelCountFix: 58151048
-  EmiMemoryLeak: 55496966
   EnableHotkeyTracing: 39486948
   EnableInteractionModeRemoting: 55372141
   EnableMSAWASCFlow: 44671690
@@ -1993,13 +1976,13 @@
   EnableSpotlightForOEMWallpaper: 59391673
   EnableSystemResourceManagerForExplorer: 31832430
   EnableUninstallSystemComponentsForDMARegions: 57623725
+  EnableUqiOnUndefinedPolicyState: 59080999
   EnableViewStatePidlNormalization_55063784: 55063784
   EnableWASCFlow: 44671686
   EnableZoneIDInRdpClip: 59646747
   Enable_55578848_Fix: 55578848
   Enable_57654056_Fix: 57654056
   EncryptedDownloadFailureRetryBug: 59767314
-  EndTask_Bugfixes2407: 50550889
   EndUserOobe_ShellHost: 59799919
   EnergySaver_HeuristicToast: 52055643
   EnergySaver_HeuristicToast_Send: 53265686
@@ -2030,6 +2013,7 @@
   ExtractProviderTypeCrashFix: 54520876
   EyeControllerApp_2508: 58276712
   EyeControllerApp_2510: 59173169
+  EyeControllerApp_2602: 60540371
   FCTSPV: 59706624
   FDConversation: 58328873
   FEListModeTextScale: 56249912
@@ -2116,7 +2100,6 @@
   FixImmediateRetryInstall: 59678013
   FixInvalidTokenHandleDuringScan: 55964290
   FixLayoutControlOverLapWithSearchBox: 56545373
-  FixLockAcquisition_UpdatePivotView: 53573094
   FixNarratorOnBulletList: 26742989
   FixNoOptionForVhdStorageOpConfirm: 59947587
   FixOobRegRead_58621379: 58621379
@@ -2138,10 +2121,8 @@
   FlashDeprecation_Notification: 25217842
   FluencyMultilingual: 19570937
   FocusSettingRejuv: 35742169
-  FocusTransitionFix_42418648: 42418648
   FocusWhenSwitchingDialogs: 56253267
   FolderIconMissingFix: 58318533
-  ForegroundOnDesktopSwitchEx: 55936567
   FormatVolumeDialogFix: 58505741
   FpSTestSm: 44713148
   FpvTest: 42187503
@@ -2192,11 +2173,9 @@
   GestureOnlyTouchpadParam2: 56107479
   GestureTargetingPTPFrameSplitFix: 53814692
   GetCreativeResultsTimeout: 56419008
-  GetDefaultAccountTokenCXHHelper: 50881018
   GetDefaultAccountTokenWXHHelper: 53284862
   GetStartedTriggerUpdates: 42963857
   GlobalDictionaryFix: 58305661
-  GraphRecentItemsManagerEnabled: 38294196
   GreenCharging: 45690917
   GuidedSnapAssistContinuity: 33793580
   HAPLS: 60067042
@@ -2206,7 +2185,6 @@
   HFLAF: 58130649
   HIDBS: 59193114
   HIDL: 60139631
-  HNSImprovementsForCilium: 57732430
   HResultForElevationDenied: 50463747
   HResultForEmptyContentCatalog: 58849030
   HSHELL_IAM_Delivery: 39786106
@@ -2226,23 +2204,14 @@
   HeaderChangesForEndpointCalls: 56620941
   HealthAttestationHandleLeak: 49140057
   HelloFaceDriverDowngradeCapability: 34933858
-  HelpLaunchTipTests_57208526: 57208526
   HideAddDeviceIfBluetoothUnsupported: 41734715
   HideBackupWizardForIneligibleAccount: 46825733
   HideKeyboardOnLogon: 58674510
   HideRecommendedSection: 36435115
   HideSystemPackages: 47989338
   HideSystemPackages_Cleanup: 51629559
-  HideSystemTrayDateTimeSetting: 41437381
   HighDrainSleepStudy: 59719389
-  HighIgnoreRateIn28337499: 57475380
   HighPowerDrainFMax: 57372467
-  HnsACIBugFixBundle: 57512508
-  HnsEndpointRoutes: 54599261
-  HnsExplicitAccessCheck: 42077418
-  HnsInternalVLAN: 49142546
-  HnsNpmPerfOptimization: 53548939
-  HnsRecreateMirroredICSResumeContainer: 40560708
   HnsWcosLoopback: 23636350
   HolographicNotifications: 9666698
   HomeDoubleFocusIndicatorFix: 58015528
@@ -2270,16 +2239,13 @@
   HydrateDuringDownload: 41666070
   ICapabilityAccess_AgileRef: 55248491
   ID45035559: 45035559
-  ID46504797: 46504797
   ID46524912: 46524912
   ID46973925: 46973925
-  ID48930609: 48930609
+  ID49059846: 49059846
   ID49780915: 49780915
-  ID50287681: 50287681
   ID51367344: 51367344
   ID51373487: 51373487
   ID51912085: 51912085
-  ID52127539: 52127539
   ID52525627: 52525627
   ID53863504: 53863504
   ID54107382: 54107382
@@ -2292,7 +2258,6 @@
   ID55815828: 55815828
   ID55986898: 55986898
   ID56134056: 56134056
-  ID56381645: 56381645
   ID56386964: 56386964
   ID56517033: 56517033
   ID56898102: 56898102
@@ -2334,23 +2299,20 @@
   IPSec_Point_To_Site: 56160586
   IPUMCPPlugin: 59576296
   ISFT: 59173225
+  ISMResetCrashFix: 48755161
   ITSNotifySinkEx: 57100252
-  IconLoadTIPFix: 46241418
   IdTest: 50203215
   IdentifySpotlightWallpaper: 58984369
   IdsBugFix1: 54100127
-  IgnoreLauncherDuplicateWebappDone: 39662864
   IgnoreOAMarkerVersion: 45387478
   IgnoreSLValueNotFoundErrorCode: 53177319
   ImageBuffer_qwdedgnide: 57073102
-  ImeActivationTest_57609483: 57609483
   ImeCompositionWindowLifeTimeTest: 59272290
   ImeFeatureUsageMeasure: 58993537
   ImeLangBarLifeTimeTest: 59256957
   ImeLangBarLifeTimeTest2511: 59720359
   ImePadLifeTimeTest: 58037679
   ImeSystemDataIconCrash: 56772229
-  ImeVersionFallbackTest_57386628: 57386628
   ImmediateRetryInstall: 57421689
   ImmersiveColorFight: 54663121
   ImmersiveShellComponentsInitializationBudget: 27075521
@@ -2371,14 +2333,15 @@
   InfraredROIControlImprovement: 31225283
   IngoreVirtualReadErrors: 57072134
   InitiateNTHPrivacyOnServer: 45067613
+  InkingWinMLReplat: 59872186
   InputDelays: 56524372
   InputDrivenEdgeAnimations: 36159954
   InputDrivenGTGAnimations: 33668320
   InputHostShutdown: 56349993
   InputHostShutdown2: 59925833
-  InputSwitcher2506: 54672880
   InputSwitcher2507: 57404898
   InputSwitcher2509: 58584215
+  InsiderPreviewBranding: 14096858
   InstallAttribution: 3222455
   InstallItemTelemetry: 58142822
   InstallServiceConcurrentInit: 60262034
@@ -2416,6 +2379,7 @@
   KBBLS_55832275: 55832275
   KBBLS_55832598: 55832598
   KBL60279530: 60279530
+  KBL60302425: 60302425
   KEKUpdateAllowList: 55813532
   KET: 58990373
   KeepWin8SearchPane: 35032763
@@ -2423,7 +2387,6 @@
   KeyTest: 50203185
   KeyboardHelperCrashAtDesturctor: 57679126
   KeyboardSettings_InputProfileHotKeys: 57979372
-  KryptonSoftLargePages: 21026262
   LCBO: 56703986
   LCCMD: 53451155
   LCFOCC: 55365939
@@ -2439,10 +2402,8 @@
   LPC_59467628: 59467628
   LREC: 60141051
   LSS: 57728035
-  LTCW: 56461365
   LWFilterOrder: 55201800
   LanguageFodSizeCheck: 55849117
-  LanguageSwitcherAnimationFix: 57629423
   LaunchDimeForAquisitionUSQRenewal: 45562151
   LaunchFileEventsAndRefactor: 52550223
   LaunchUriForResultsMeasureEvents: 52937186
@@ -2453,9 +2414,7 @@
   LicenseManagerShadowAdminOnesettingsCheck: 59112334
   LightThemeAccentColorRedirectionForRail: 47326335
   LightingWhenLidClosed: 16487091
-  LiteWhiteNTH: 33355279
   LoadBloomCreatives: 45636699
-  LocalizedTipsLinkOnPen: 56533847
   Location_Fix_GeofenceDeleteTypeMismatch: 59955951
   Location_Fix_GeofenceTrackerDisposeOrder: 47787036
   Location_Fix_GnssAdapterDeadlock: 57948082
@@ -2475,7 +2434,6 @@
   Location_OrionSourceBreakdown: 60009833
   Location_TokenCacheFallback: 59807255
   LockBugFix_LockAppInstantlyDismisses: 20191093
-  LockInNativeBridge: 57318141
   LockOnLogon: 32979348
   LockPolicyImprovementForCTA: 54880975
   LocklessPhasesPoCallout: 39365653
@@ -2487,12 +2445,10 @@
   LogonSound_EndpointReadiness: 54704131
   LogonTabTipGroupPolicy: 60234620
   LogonTaskDurationsComputationFix: 58632488
-  LogonTaskZeroDurationsFix: 55347275
   LogonTouchKeyboardCheckResources: 58950676
   LogonTouchKeyboardNotLaunching: 58635597
   LogonUIDoNotContinueAfterCrashing: 57311441
   LogonUINoUpdateUserInput: 30723191
-  LpacComProxyAce: 46549834
   LwhDisallowMicrosoftCorporation: 58306888
   M365CopilotPinning: 57512414
   M365CopilotPinningFrequency: 57575939
@@ -2506,7 +2462,6 @@
   MDMErrnoFixes: 58321559
   MDM_Extensibility_DSC_Native: 37252711
   MDM_Fix_WinDC_Alert_Size: 42280597
-  MDW: 46702397
   MFACheckInClipRenew: 38124097
   MFLAsync: 42877930
   MIDI2: 38679741
@@ -2543,7 +2498,6 @@
   MercuryBackupValidationExecutor_Obsolete: 59351901
   MicaBackdropInApplicationFrameHostTitlebar: 35681203
   MigrateAppCategoryMappingsCache: 56361829
-  MigrationProgIDsBugFix: 46879732
   MinMPOSize: 56296367
   MinorCrashFixes2508: 56594280
   MinuteZeroExperiences: 8497738
@@ -2569,7 +2523,6 @@
   MsfteditGB_2022A_BugFix: 55953452
   MsiHashMismatchFix: 33280101
   MsiReinstallModeOverrides: 45317952
-  MstscUninstallSupport: 0
   MtfClientHolderBugfix: 54448755
   MultiDetachedOC: 46440044
   MultiRebootUpdateAndShutdownFix: 57300273
@@ -2616,16 +2569,15 @@
   NDPQualitySummer25: 57263685
   NDTAF: 58398053
   NDUPCSPBridgeAPI: 59966429
-  NDUPEarlyOobeNode: 49292347
   NDUPLanguageRemoval: 59282748
   NDUPLanguageRemovalV2: 60131629
   NDVF: 60335299
   NEABF: 57607603
   NEDAO: 59924607
+  NEMEF: 60611859
   NIDRF: 56918607
   NLSF: 55673107
   NMCI: 59249202
-  NNVDTCF: 56112207
   NNV_AVDA: 60389897
   NNV_DVBSV: 58416033
   NOLBF: 54183071
@@ -2634,11 +2586,14 @@
   NPFW: 59730510
   NPNL: 59681391
   NPUDetect_EnableCloudNPU: 57914358
+  NRSB: 60604975
   NSFTI: 59450295
   NSLTF: 59154154
   NSRSC: 56975771
   NSRUI: 57800595
+  NSSBF: 60073289
   NSTDNI: 42810361
+  NTCWC: 54566904
   NTF: 57713327
   NTSA: 58541741
   NUT: 55729817
@@ -2735,8 +2690,8 @@
   NotifyInputUIMessageTryCatch: 45656204
   NotifyProcessCreateAlways: 56757779
   NpuMultiNodeEnergyFix: 55723466
-  NthM365Pinning_ShellHost: 59721088
-  NthWasc_ShellHost: 59697487
+  NthM365Pinning_ShellHost: 60555849
+  NthWasc_ShellHost: 60555845
   NullAccountCrashFix: 57468531
   NullTerminateLayoutFileData: 59113381
   NxtKspTransaction: 55061615
@@ -2745,12 +2700,12 @@
   ODRSTF: 56121346
   ODTTVA: 55442377
   OEMCustomImagesAreDesktopSpotlightImages: 48696782
+  OEMThemeCheckTiptest: 60491180
   OOBEDiagnostigsCleanupFix: 57168260
   OOBEOneDriveShellExperience: 43121389
   OSAF: 56918265
   OSK_MAD: 60248685
   OSK_SecureOnLockScreen: 44583694
-  OTPGTB: 38653309
   ObeyHideAllAppsPolicy_56396958: 56396958
   OemEsimStore: 29834243
   OfflineLicenseMigration: 54159450
@@ -2769,20 +2724,13 @@
   OnlineEFS: 8167107
   OnnxColorFaceDetector: 35780056
   OnnxInfraredFaceDetector: 42429607
-  OobeBloomUI: 46450614
-  OobeDisableAltF4: 47409595
   OobeExperienceManageQueueWnfCallback: 26243368
   OobeForceFeatureConfigurations: 48813707
   OobeHelloNarrationFixOnConfirmation: 58964371
-  OobeHostAppEnvironmentCleanupFix: 58353102
   OobeHostAppInDefaultUserSession: 54823363
   OobeHostNetworkReconnect: 53959297
   OobeHostReconnectCaptivePortal: 59214822
-  OobeHostRetrieveCXHSharableData: 48898165
   OobeHost_SupportRecallConsent: 57641161
-  OobeIntroVideoSynchronization: 53460715
-  OobeKioskPasswordlessAutologonFix: 56883565
-  OobeLaunchCommandPromptToForeground: 57954736
   OobeListViewKeyPressSearch: 54847396
   OobeNaviNameFix: 56021041
   OobeNdupUsoTask: 48126282
@@ -2790,12 +2738,11 @@
   OobePC2PCMigration: 55111207
   OobePersonalizedOffers: 48142773
   OobeRestoreDataLayer: 43933049
-  OobeRestoreUx: 43187998
-  OobeSetBackgroundImage: 58487060
   OobeShellHostWin32App: 52601469
   OobeShellHostWin32App_Wave2: 54828955
   OobeWebAppTelemetryFix: 57704319
   OobeWindowsAccountSyncConsentPage: 44406154
+  OobeWizardNullCrash: 60445073
   OobeWv2HostDll: 53284810
   OobeWv2HostDll_FindWinAppSDKVersion: 59831603
   OobeWv2HostDll_Wave2: 56299140
@@ -2820,11 +2767,13 @@
   PATKTI: 37007953
   PCA2023BootMgrUpdate: 51896780
   PCAIDSIntegrationFix: 52452917
+  PCBF: 60664360
   PCPKspImportPkcsFix: 56051438
   PC_TO_PC_Migration_App: 48753452
   PDCBugBundle_2509: 58336771
   PDE_HAADJSupport: 44924266
   PFVC: 42187469
+  PI: 60640664
   PLButtonNarratorFix: 57184077
   PMVV: 56309835
   POIrisClients: 49203781
@@ -2833,7 +2782,6 @@
   PairedLocaleInputMethods: 36942819
   ParallelizeInspect: 39662810
   PartialProcessLimit: 50303602
-  PassCallerUriArgs: 50880979
   PassedComponentName: 58005039
   PasswordAutofill: 13944250
   PauseAutoAppUpdates: 57853873
@@ -2852,8 +2800,7 @@
   PdfTileCache: 12976578
   PenDetachIWS: 20430744
   PenWorkSpaceAutoHideInteropFixes: 59301057
-  PerUserSetup_MultiFlow_WNFTimingFix: 57467091
-  PerUserSetup_TIPIntervalDetection: 56096818
+  PerFolderRootPropertyTestNoise: 58954450
   PerfTrackSummary: 57412844
   Pers_PreviewControl_MinSize_Fixes: 59228011
   PersonalizationCustomThemeBackupHandler: 45733106
@@ -2879,6 +2826,7 @@
   PnpUtil_AdditionalCatalogAttributeLogging: 60235998
   PnpUtil_EnumCatalogAttributes: 55755652
   PnpUtil_EnumPreprodCatalogAttribute: 57466877
+  PnpUtil_LogBlockedDrivers: 58302668
   PoAdaptiveStandby: 49614451
   PolicyCacheRefreshTelemetryWSUSHash: 51602840
   PollDownloadProgress: 60159359
@@ -2890,7 +2838,6 @@
   Prefer2kEkOver3k: 56831420
   PreinstalledAppSupport: 35286366
   PreloadWinners: 41515636
-  PrepinTFL: 42560982
   PreserveSecureAssessmentApp: 18212771
   PreventRepeatedAdminChecks: 55049146
   Print: 13944184
@@ -2901,6 +2848,7 @@
   ProcessIrisActionsStateChangedWNF: 35431603
   ProfanityPostFilter: 27864046
   Professional_BaseImage: 38746023
+  Profile_Service_ETW_Logging: 46832940
   Profile_Service_Retry_Load_Loop: 24172318
   ProgressiveCalistaSizeCheck: 59166055
   ProtectUserDictionaries: 29674961
@@ -2948,7 +2896,6 @@
   RDX_TimedShutdown: 6523167
   RDX_VirtualDesktopCleanup: 44848897
   RFBEFR: 56394035
-  RFS: 52966583
   RH1S: 0
   RH5S: 0
   RLTA: 53467876
@@ -3003,7 +2950,6 @@
   RemoveActivityPrivacyPage: 53036850
   RemoveAdID: 44517656
   RemoveApartmentContexts: 58390198
-  RemoveChat: 43088148
   RemoveDamPhaseForHam: 56188665
   RemoveDeviceConfirmatorDeprecation: 34525664
   RemoveEdgeEnforcementOnHotspots: 45049929
@@ -3022,8 +2968,6 @@
   Remove_Thin_Space_blocker: 49560522
   RemovingMailAndCalendarAppSetting: 56277800
   RenewTelemetryForBrokenKeys: 58744036
-  ReorderOobeLanguagePageSkipConditions: 53176946
-  RepairAppRegistration_Preserves_ApplicationData: 42969627
   RepeatedStorageSettingsDiskAndVolumesUACs: 56772405
   ReplaceDeprecatedGetTempPathAPI: 56360129
   ReplaceDictationOverlay: 58978959
@@ -3054,7 +2998,6 @@
   RobocopyEfsRawNoRetryFix: 55739196
   RollbackForAtomicTransactions: 34643634
   RotateFrameToDetectFace: 26514007
-  Rtmep: 31530550
   RulesEngine: 41121012
   RunSynchronousOptionForRedirectExeShim: 0
   SAC_ClientSideEnable: 57880660
@@ -3062,13 +3005,12 @@
   SAC_Wldp_Provider_Register_Bugfix: 54474375
   SBDFESSOA: 0
   SBDFESSOD: 0
-  SC: 57253914
   SCCRedirectionTrustPolicy: 59209871
   SCE: 57726743
   SES: 53338564
   SH: 53252471
-  SHINE_2502_Bugs: 53719087
   SHINE_2509_Bugs: 58553859
+  SHINE_2602_Bugs: 60522487
   SIVS: 45356343
   SKAS: 50210837
   SMARS: 55999643
@@ -3077,8 +3019,8 @@
   SafeEventRegistration_V2: 58554729
   SafeModeTouchKeyboard: 59156855
   SafeRegistryWatcher: 51575430
+  SandboxCopyImmutableKeys: 59684366
   SavePasswordSoftlandingEvent: 12261764
-  ScanAllAltSecIds: 59794829
   ScanForUpdatesOnServer: 46340594
   Scenario55536346: 55536346
   Schannel_STSTimeBugfix: 46700378
@@ -3098,8 +3040,6 @@
   ScriptingEngines_DisableLegacyJScript_ProcessLevel: 27806502
   ScriptingEngines_DisableLegacyJScript_SystemLevel: 26092979
   ScriptingEngines_GroupPolicyJScriptReplacement: 31916716
-  SearchBoxOnTaskbar: 40887771
-  SearchBoxOnTaskbarAutoHide: 49287050
   SearchBoxOnTaskbarShyTaskbar: 49559611
   SearchFeedbackStringFix: 60334369
   SearchInAllMarkets: 17937263
@@ -3132,13 +3072,13 @@
   ServicingNormalPri: 57200960
   ServicingTimeBackwards: 51171417
   Servicing_37313118: 37313118
+  Servicing_AADSidToNameV2Support: 60314197
   Servicing_ACRMCRRepairingBackupManifests: 60256417
   Servicing_ACRNotTriggeredDuringSingleHopforMissingFiles: 59730077
   Servicing_AEBBSTRLeak: 56588631
   Servicing_AEBBSTRMultiString: 60451329
   Servicing_ALPCServerShutDownFix: 21623808
   Servicing_ARBECDB: 58676768
-  Servicing_ARBOffGlobal: 57397161
   Servicing_AccessAgentItem: 55735922
   Servicing_AccessibleNameNoThrow: 55418223
   Servicing_AddFullTimestampsToGPOLogs: 58026485
@@ -3154,7 +3094,6 @@
   Servicing_AppVOfficeAppsDoubleHandleCloseCrash: 59708755
   Servicing_ApplyNoExceptGuardForVolumeRemove: 45186820
   Servicing_ArbiterOfflineImageArch: 56464558
-  Servicing_Arm64SingleInstancing: 57067858
   Servicing_AsyncAllowDrop_47476131: 47476131
   Servicing_AsyncMFTWrapper_FormatChangeLostOnFlush_Fix: 0
   Servicing_AttributeContentNameMSRC92108: 54631797
@@ -3174,7 +3113,6 @@
   Servicing_BaselineAcquisitionFix: 49668334
   Servicing_Bcastdvr_NoFrameDurationCheck: 59254682
   Servicing_BfsvcReboot: 58950549
-  Servicing_BindRequest_UE: 56288992
   Servicing_BlockHtaInPEFile: 58127661
   Servicing_BlockSecureBootUpdate_OlderSystems: 58022765
   Servicing_BluetoothUX_RadioWatcherExceptionFix: 57288874
@@ -3204,13 +3142,12 @@
   Servicing_CBSPBRLCUWrapper: 56934864
   Servicing_CBSPSC: 59218460
   Servicing_CBSPSRLImprovement: 57836102
-  Servicing_CBSParsed: 58023476
   Servicing_CBSPlan: 58761982
   Servicing_CBSPlan2: 59291529
   Servicing_CBSRestartHandling: 50204381
+  Servicing_CBSSBU: 59948262
   Servicing_CBSSC: 55387220
   Servicing_CBSScavengeCorruption: 54162663
-  Servicing_CBSTInit: 57069194
   Servicing_CBSUPL: 54480625
   Servicing_CBS_CCBSPD: 58675746
   Servicing_CDPAccessviolation: 55385446
@@ -3224,16 +3161,13 @@
   Servicing_Castguard_javascriptstackwalker: 48038168
   Servicing_CategoryOverflowNarratorFix_57276123: 57276123
   Servicing_CbsIPUFix: 58301402
-  Servicing_CbsTempRemovalFix: 57900857
   Servicing_CertificateTemplateCpsUrlOidIssue: 56841720
   Servicing_CgFunctionBody: 56457562
   Servicing_CgPathTypeHandler: 56456883
   Servicing_ChakraOOB: 55960114
   Servicing_CleanupIEEntrypoints: 55948192
   Servicing_ClipCryptoLeak: 59974269
-  Servicing_CloudBMRWiFiAutConnect: 56369455
   Servicing_CloudFilesSearch_CloudProviderFindTipTest: 56062274
-  Servicing_CloudFilesSearch_ExternalDataCollectorRelease: 56510036
   Servicing_CloudFilesSearch_GalleryFix: 55918674
   Servicing_CloudFilesSearch_GalleryNavigate: 55888583
   Servicing_CloudFilesSearch_GroupedResults: 56159287
@@ -3251,6 +3185,7 @@
   Servicing_CodeSign_Verification_Telemetry: 56881338
   Servicing_ColorManagementFilePickerUpdate: 52147586
   Servicing_CommandBarStealsFocusOnShiftF10: 44221343
+  Servicing_CompanionsDMAFix: 60725592
   Servicing_CompilerWarningC4146Issue: 59359761
   Servicing_CompositionChangeTransmog: 60058165
   Servicing_CompressedAudioPassthrough_EnhancedFormatChange: 0
@@ -3261,11 +3196,10 @@
   Servicing_Containers_DisableChpeForHotPatching: 51721202
   Servicing_Containers_MapHostCatalogs: 50916384
   Servicing_ContextMenuLabelsCrashFix_55550636: 55550636
-  Servicing_ContextMenuPrimaryCommandLabelsFix_49033481: 49033481
   Servicing_ControlPanelNameNotFoundGB18030: 60083542
   Servicing_ControlPanelResizeFix: 46018710
-  Servicing_CopilotC2Pivot: 52127557
   Servicing_CopilotDesktopIconsFix: 47069058
+  Servicing_CredUIBroker_Packaged_Mip_Workaround: 60751997
   Servicing_CredUIRejuvFixMissingPasswordIcon: 60432188
   Servicing_CrossDomain_NameSid_Lookup_Fix: 59344967
   Servicing_CtacStoreAppsVer: 43882673
@@ -3289,7 +3223,6 @@
   Servicing_DeviceManagementMdmdiagnosticsEmptyRegMultiSzCrash: 51934651
   Servicing_DeviceQuery_DeserializeCDPDeviceInfo: 56158306
   Servicing_DialogChildFocusFix: 48132628
-  Servicing_DisableAllCopilotEntryPoints: 50223223
   Servicing_DisablePumpTimerForAcxEndpoints: 58480230
   Servicing_DoNotSkipAuthInWinREForBitLocker: 56381695
   Servicing_DocumentDeletionTransientErrorHandling: 53952058
@@ -3300,7 +3233,6 @@
   Servicing_DumpWinREVersion: 59950321
   Servicing_ECCSmartcardLogonCache: 58638439
   Servicing_EDAMCSP_OrchestrationFixes: 55543942
-  Servicing_EFISystemPartitionSize: 55127245
   Servicing_EO14117_IsNonUS: 57170002
   Servicing_Eaphost_TvsFixes: 60600404
   Servicing_EcoQosDownlevel: 58695640
@@ -3323,7 +3255,6 @@
   Servicing_EstablishedTrustAnchorTrust: 51992834
   Servicing_Excessive_SysvolDC: 54744879
   Servicing_ExpediteNextReboot_49811837: 49811837
-  Servicing_ExpeditedUpdateWindowsCSPPolicy: 55578570
   Servicing_ExtMonStatusSPR: 57166297
   Servicing_ExtendedAuditingForKerberos: 53662408
   Servicing_ExtendedAuditingForKerberos_V2: 57835336
@@ -3369,9 +3300,6 @@
   Servicing_FileExplorerCustomUIAProvider: 36841606
   Servicing_FileExplorerDehydratedThumbnailsChangeNotificationsFix: 54761410
   Servicing_FileExplorerDehydratedThumbnailsRenderFix: 54412425
-  Servicing_FileExplorerFixAddressBarInvalidPointers_45106222: 45106222
-  Servicing_FileExplorerGallerySerialization_46029533: 46029533
-  Servicing_FileExplorerGdiLeaks_41854967: 41854967
   Servicing_FileExplorerHomeDefCollectionLeak_45978227: 45978227
   Servicing_FileExplorerHomeNetworkPathHang: 44304240
   Servicing_FileExplorerInvalidBatchTimer: 45945617
@@ -3380,12 +3308,9 @@
   Servicing_FileExplorerPerf_BroadcastSingleMessage: 46084421
   Servicing_FileExplorerRibbonFallbackFix: 40012896
   Servicing_FileExplorerScaffoldingWindowFix: 42492233
-  Servicing_FileExplorerSearchDispatcherTimerLeak_50712415: 50712415
   Servicing_FileExplorerThemeChangeFix_46028495: 46028495
   Servicing_FileExplorerToolTipSurrogateTruncationFix: 58576380
-  Servicing_FileExplorerViewLocationNullptrFix_48004377: 48004377
   Servicing_FileExplorerWinAppSDKCachedState: 46188883
-  Servicing_FileExplorer_CancelConfirmationDialogIgnoredFix: 58449057
   Servicing_FileExplorer_ContextMenuAsyncTipCrash: 58348806
   Servicing_FileExplorer_CopyEngineNarratorFix: 51165104
   Servicing_FileExplorer_DwmCrashFix: 48207748
@@ -3426,7 +3351,6 @@
   Servicing_FodTelemetry: 55321020
   Servicing_FolderOnDemand_FreeUpSpaceFix: 47113245
   Servicing_FolderOnDemand_SearchHydrationFix: 47113215
-  Servicing_FolderViewBrowserNavigateMouseButton_46695836: 46695836
   Servicing_ForEachStreamHang: 57326442
   Servicing_ForcePSRL: 51880205
   Servicing_ForceSanitizationOfEndpointId: 59983138
@@ -3470,18 +3394,14 @@
   Servicing_IcMWithRecentMsuChanges: 57802524
   Servicing_Ignore_Failed_Op_Stream_Events: 0
   Servicing_IndexeddbMultiThreadFix: 44026119
-  Servicing_InfoToolTipManagerWeakRefCrash_45210081: 45210081
   Servicing_InputService_hProfiles_memory_leak: 56904185
-  Servicing_InspectBugFixes: 55950532
   Servicing_InsydeClevo6BDBXBootFailure: 58054496
   Servicing_IntegratedServicesPolicyForW365AndAVD: 56967096
-  Servicing_InternalL2LayerMissing: 55465894
   Servicing_InternetAvailability_Telemetry: 59669901
   Servicing_InternetDisconnectFix: 43313878
   Servicing_InvalidAddress_PhysicalDiskRefresh: 59372710
   Servicing_IsRequestingAttentionNoThrow: 58317844
   Servicing_Itss_Out_Of_Bound: 55790841
-  Servicing_JpegLargeBufferFix: 56461584
   Servicing_JpnIhdsMemoryLeakFix: 51617095
   Servicing_KEK_NoReboot: 59643749
   Servicing_KSOPMUtilRSAFix: 58595519
@@ -3500,13 +3420,11 @@
   Servicing_KeyboardFocusIndicatorIssue: 56624315
   Servicing_KeyboardFocusIndicatorListHeader: 57346705
   Servicing_KeyboardFocusSortIcon: 58525987
-  Servicing_LPRemove_Block_CJK: 56861090
-  Servicing_LampArrayHidTelemetry: 46269702
-  Servicing_LampArrayPowerPerf: 46936193
+  Servicing_LapsOrphanedAccount_Fix: 59795899
   Servicing_Laps_RestrictedGroups_crash_fix: 57900029
   Servicing_LargePage_Unique_MultiValues: 59439651
   Servicing_LastBundleTelemetry: 57706468
-  Servicing_Ldap_ValueParsing: 59775092
+  Servicing_Ldap_ValueParsing: 55794630
   Servicing_LeastSetBitC4146Fix: 55578844
   Servicing_LeftAlignment: 58356013
   Servicing_LegacyKoreanIME: 54217113
@@ -3515,7 +3433,6 @@
   Servicing_LogCompression: 57261950
   Servicing_LogicPipelineResolver: 54852096
   Servicing_LostSampleMappingDuringRateSet: 51184607
-  Servicing_MDEGetImpersonationTokenFix: 39148892
   Servicing_MDELogicV1MetricReporting: 57712665
   Servicing_MDEPayloadCapabilities: 53836691
   Servicing_MDERestartableRegWatcher: 44531195
@@ -3556,7 +3473,6 @@
   Servicing_MiracastAudioFormat: 58772781
   Servicing_MiracastDisplayID: 54867164
   Servicing_MiracastUibcLeak: 57664115
-  Servicing_MissingMum: 57723843
   Servicing_ModernDeployment_ClearUserEspCacheOnComplete: 56035478
   Servicing_ModernDeployment_DppRebootFix: 56035571
   Servicing_ModernDeployment_DppResilience: 56036989
@@ -3579,7 +3495,6 @@
   Servicing_NoDesktopTrayFix: 48446971
   Servicing_NoGenericRetryForUIAGetSelectedText: 42327107
   Servicing_NoIdentifyLevelPacValidation: 51718582
-  Servicing_NoThumbnailsIItemVersionMismatchFix_45999747: 45999747
   Servicing_NodeMgrCompilerWarningC4146: 55898029
   Servicing_NoisyAepException: 56112280
   Servicing_NonSec_FuzzingBugWindowsException_USER_RtlpHeapFatalException: 59903825
@@ -3595,9 +3510,7 @@
   Servicing_NullCheck_For_LanguagePack: 57086413
   Servicing_NullCheck_HostChannelManager_AuthProvider: 56101556
   Servicing_NullPointerSafety: 56457081
-  Servicing_OCCC: 57540106
   Servicing_OCMFL: 57023321
-  Servicing_OCNR: 56814346
   Servicing_OOBReadFix: 56240137
   Servicing_OSK_LockscreenCheck: 45174402
   Servicing_OdbcCPReturnDeadCon: 43761715
@@ -3614,7 +3527,6 @@
   Servicing_PABNarratorACDC: 55814650
   Servicing_PBRAlwaysCopyPanther: 58777925
   Servicing_PBRArmOSPieces: 34298243
-  Servicing_PBRDetectCorrectWinRERevision: 57372834
   Servicing_PBROnlineInitCatch: 58290847
   Servicing_PBRPS: 57479096
   Servicing_PDEPage: 47976101
@@ -3631,7 +3543,6 @@
   Servicing_PackageContainmentTelemetry: 58261001
   Servicing_PasswordReset_WaitForLockAppHost: 56999576
   Servicing_PaygoUpgradeRemoveRegistry: 59888449
-  Servicing_PayloadRemovalInResolve: 57002583
   Servicing_PdfFuzzingIssue: 56409040
   Servicing_PendingSmartFeatureFix: 50181614
   Servicing_PfxImport_EmptyInstanceData: 59799315
@@ -3648,8 +3559,6 @@
   Servicing_PrintManagementDataPersist: 54890316
   Servicing_Print_SpoolerDeadlockFix: 58676754
   Servicing_Print_ZhCn_LanguageCompatibility: 59258066
-  Servicing_ProcessRequestEx_UE: 56428208
-  Servicing_ProgrammableTaskbarRunOnceCriteria: 49253617
   Servicing_ProtectMediaBreakMarker: 58528201
   Servicing_ProtectedUsersBlock: 53798507
   Servicing_QMRAutoPatchDefault: 59025836
@@ -3681,6 +3590,7 @@
   Servicing_RecoveryRobustness_OnlineWinREProvisioning: 57665022
   Servicing_RecoveryRobustness_RemoteManagement: 55936146
   Servicing_RecoveryRobustness_SettingsPage: 53497692
+  Servicing_RecoveryRobustness_SetupRecoveryDataTask: 60337618
   Servicing_RecycledContainerTypeMismatch_56119816: 56119816
   Servicing_Reduce_Codec_Pack_License_Checks: 60193066
   Servicing_RefreshGmsaPasswords_Fix: 50205382
@@ -3690,16 +3600,15 @@
   Servicing_RemoteCredGuardPDUs: 55200751
   Servicing_RemoteRemediationCSP_IntuneAtomic: 59820021
   Servicing_RemoteRemediationNetworkPassword: 58704307
-  Servicing_RemoveSearchDelayTimer_55738909: 55738909
   Servicing_ReportAccurateCameraError: 0
   Servicing_ResetOptionalBrowserExtensions: 45236164
   Servicing_ResetRecommendedAppFix: 42868018
   Servicing_ResilientCorruptionList: 59154284
-  Servicing_RogueRunningIndicator_57247657: 57247657
   Servicing_RpcBufferAuditing: 56550930
   Servicing_SAR_EndpointSwitchDebounce: 0
   Servicing_SAR_OffloadFallbackOnLowResources: 0
   Servicing_SAR_StarvationRetryLoop_Fix: 0
+  Servicing_SBAITickler: 60569158
   Servicing_SBKeyRoll_10B: 59266774
   Servicing_SBKeyRoll_8B: 57836222
   Servicing_SBKeyRoll_Search: 59284462
@@ -3737,7 +3646,6 @@
   Servicing_SearchUXHighContrastBorderFix_55802560: 55802560
   Servicing_SearchUXIconNonIndexedLocationFix_55656186: 55656186
   Servicing_SecHandleAlignment: 58552301
-  Servicing_SecondaryTaskbarUpdateSize: 49645819
   Servicing_SecureBootKeyRollPolicyFix: 59713297
   Servicing_SecureBootOA3ToolTelemetry: 57834508
   Servicing_SecureDesktopBugFixes_48245029: 48245029
@@ -3752,7 +3660,7 @@
   Servicing_SetToolTipIsOpenReliability_51787449: 51787449
   Servicing_SetupDiskBackButton: 56333376
   Servicing_SetupFixiSCSIDeployment: 52577977
-  Servicing_SetupPlatform_WSBStablePath: 56485705
+  Servicing_SetupPlatform_WinSetupBootVolumeIssue: 58446257
   Servicing_SfcDisableHardErrorMsgBox: 56257997
   Servicing_SharedPCRetryMechanism: 57245335
   Servicing_ShdocvwAuthenticodeCheck: 43099179
@@ -3772,7 +3680,6 @@
   Servicing_StopRTMBaselineAcquisition: 58263318
   Servicing_StopSearchUXAnimationsWhenTurnedOff_55384482: 55384482
   Servicing_StopTimerOnDestructor_57359035: 57359035
-  Servicing_StorageItemFastFlag_56455181: 56455181
   Servicing_SubauthLsassMemoryLeak: 59380830
   Servicing_SuppressCorrectInboxPackageCheckForSystem: 57383490
   Servicing_TFLPrePinMDMFix: 46645502
@@ -3780,24 +3687,16 @@
   Servicing_TaskbarAepHoverInvoke: 58568722
   Servicing_TaskbarApartmentContext: 38194581
   Servicing_TaskbarBackgroundCrash_55377849: 55377849
-  Servicing_TaskbarClipped_44356656: 44356656
   Servicing_TaskbarDarkPlateFix_41062168: 41062168
   Servicing_TaskbarExtendedUIGhostWindow: 58993528
   Servicing_TaskbarFrameTabStop_50881396: 50881396
-  Servicing_TaskbarIconLoadPending: 48047560
   Servicing_TaskbarImmersiveShellPinFailureFix: 49147730
   Servicing_TaskbarInitializationTest: 54264929
-  Servicing_TaskbarLegacyTeardownCrash: 41836164
-  Servicing_TaskbarModelTabletPostureSaturatesThreadPool: 43799312
   Servicing_TaskbarMultimonForeground: 44560958
   Servicing_TaskbarNarratorPinnedItems_48093475: 48093475
-  Servicing_TaskbarNarratorUnpinnedItem: 48226192
   Servicing_TaskbarOverlayIconFlickerMRU: 58265455
   Servicing_TaskbarPinRQVFix: 56396077
-  Servicing_TaskbarRemoveIconMirroringRTL_57196189: 57196189
-  Servicing_TaskbarRpcFailfastFix: 44756417
   Servicing_TaskbarShiftRightClickCrash_38343607: 38343607
-  Servicing_TaskbarSnapGroupOnItemResolved: 49344322
   Servicing_TaskbarSnapGroupPendingItemList: 55284361
   Servicing_TaskbarTabsInNewTaskGroup: 58400622
   Servicing_TaskbarUnpinFallbackForJumplist: 55354766
@@ -3806,8 +3705,6 @@
   Servicing_TelemetryStringFixInPlaceholderCode: 53256044
   Servicing_TelibFixRetryLogic: 55713571
   Servicing_TelibTrustAnchors: 52321961
-  Servicing_ThumbBarFlickering_55535676: 55535676
-  Servicing_TicklerRebootSupport: 60515167
   Servicing_ToastCenterAccessibleNameWatson: 45053614
   Servicing_ToshibaAsusFirmwareblockList: 58761871
   Servicing_TransmogMissingPackageFix: 49792578
@@ -3842,7 +3739,7 @@
   Servicing_VAMCoRegisterDeviceCatalog: 46110737
   Servicing_VBScript_Enhanced_Logging: 57540041
   Servicing_Validate_Codec_Packs_OOP_For_System: 60090767
-  Servicing_VelRebootLogging: 59342500
+  Servicing_VelRebootLogging: 60760568
   Servicing_VelRebootLoggingSRM: 60470936
   Servicing_VelRestartProfiles: 55344197
   Servicing_Virtual_MOTW_Copy_Fix: 56182057
@@ -3888,7 +3785,6 @@
   Servicing_Wow64DefenderClientLoad: 51254211
   Servicing_WpfCrashInTransitoryDocument: 55752217
   Servicing_X86StackFrame: 57459839
-  Servicing_XAMLViewOnLoadedEventFix_46617862: 46617862
   Servicing_XHEAACPeakLimiter: 59459198
   Servicing_XHEAACSbrDelay: 60180695
   Servicing_XInputHidWGIRace: 58485302
@@ -3907,6 +3803,7 @@
   SetWindowBandViaBroker: 58604297
   SettingCommandSearchBoxFlyoutMinWidthBugFix: 60315100
   SettingCommandSearchBoxMarginBugFix: 58448164
+  SettingCommandSearchResultsIconMarginFix: 59837052
   SettingsAIComponentsPage: 47688503
   SettingsAIProfileImage: 47364920
   SettingsAccountsInfoProgrammablePromoCard: 12908183
@@ -3932,8 +3829,10 @@
   ShareAnyWindow_VisualIndication: 35232263
   ShareOnPdf: 14434234
   SharedPCCSPImprovements: 7179030
+  ShellAppRuntimeAsShellProcInsteadOfRdpInit: 0
   ShellHostInBasicJobObject: 59989826
   ShellHostPinning: 58291001
+  ShellHost_DisableUriCheck: 60555868
   ShellUIHostCoordinatorSandboxCheck: 57437795
   ShellUIHostCoordinatorSkuCheck: 56120340
   ShortUserPresenceTimeout: 51247650
@@ -3947,11 +3846,9 @@
   ShowSmartInstallAppRecommendation: 34907890
   SignInOptions_19H1Page: 14575813
   SilentInstalledApps: 8025889
-  SingleEnduserOobeApp: 51776237
   SinglePathAppRestart: 32060401
   SinglePointModePinnedFoldersV2: 57246742
   SinglePointModeRearrangePinned: 50472543
-  Siscs: 41314201
   SkipCanAttemptUpdateAfter_Wait: 56841513
   SkipCloakedWindowsWhenFallingBackToLastActiveWindow: 36556458
   SkipDeviceCheck: 57174863
@@ -3960,10 +3857,11 @@
   SleepOffenders: 59993183
   SleepStudySlimControlEtl: 39487788
   SlowAppLaunch: 56524389
+  SlowAppStopOnly: 60682900
   SmartActionsFlyout_AFX: 55797166
   SmartActionsFlyout_ActionsUI: 55960886
   SmartPrint: 13476978
-  SmoooothEduSharedPCOOBE: 0
+  SmoooothEduSharedPCOOBE: 6988424
   SnCm: 32966687
   SnCmEnablment: 31979781
   SnapBarDiscoverabilityInlineMessage: 53655173
@@ -3997,8 +3895,6 @@
   StartMenuAccountBadging_ColorFix: 49194827
   StartMenuLogonTaskTipTest: 58497514
   StartVersionTelemetry: 57785534
-  StateRepository_StartupIndexesOfflineUupProductChanges: 51996460
-  StaticCloudNavmesh: 42421716
   StayQuietAfterSuspend: 57397170
   StereoDFlip: 57045438
   Sticker2506_Improvement: 57211122
@@ -4007,7 +3903,6 @@
   StopWebAppTipTestOnNavigationFailed: 55952497
   StorSvc_BitLocker_Home_SKU_Fix: 53691571
   StorSvc_DeviceSize_UNC_MSRC_Fix: 56377750
-  StorageAPIsJsonObjectSupport: 49576148
   StorageAdminAltDataModifier: 57398187
   StorageItemPrivacyChecks: 13848133
   StorageLibraryFolderSuggestions: 9004739
@@ -4028,6 +3923,7 @@
   SuggestedChatAgentPromptProvider_JumpList: 58419385
   SupportCDMInDefaultUser: 9300179
   SupportPenInvokeAlways: 57937573
+  SupportPenInvokeAlways2602: 60326573
   Support_AzureEdition_On_Air_Gapped_Clouds: 58980788
   SuppressSACSplashScreenAndTitleBar: 57602257
   SurfaceWUUpdateId: 57618640
@@ -4070,7 +3966,6 @@
   TPMDriverMemoryTISWriteFix: 58097621
   TRABF: 56610030
   TSM: 55954321
-  TUC: 29534009
   TUPF: 56343439
   TVS459: 54903677
   TabTip2511Updates: 59896820
@@ -4085,7 +3980,6 @@
   TailoredScoobeM365SubscriptionFlow: 43133914
   TailoredScoobeODSignUp: 51461459
   TailoredScoobePCBackup: 43231896
-  TailoredScoobeWXH: 46008647
   TailoredScoobe_ShellHost: 60128001
   TargetedContentAPI: 8299381
   TargetedContentCacheExpiration: 11946054
@@ -4093,8 +3987,6 @@
   TargetingSessionContextV6: 8049662
   TaskManagerExperiment: 8749962
   TaskManagerServerIsClient: 47455341
-  TaskbarCopilot: 44774629
-  TaskbarCopilot_Settings: 44850061
   TaskbarDefaultFocusOnStart: 49134392
   TaskbarDefaultIconFallback: 49683902
   TaskbarDynamicIconScaling: 29785184
@@ -4135,7 +4027,6 @@
   TipTestCentennialLaunchUriForResultsImprovements: 54121714
   TipTestForSpotlightNotOnboardedReasonForLogonUpgrade: 59683216
   TipTestForUndockedSpotlightNotEnabled: 59669157
-  TipTsfAddWordTest_57373107: 57373107
   Toast: 53475297
   ToastActivationForeground: 57987584
   ToastsInSlui: 50065219
@@ -4166,7 +4057,6 @@
   TrimWithLock: 58288541
   TryCatchTakeForeground: 58010678
   TryShellActivateApplicationFallback: 23674478
-  Tsf3ImeExperienceLogon: 0
   TurbostackFallbackToBaselineHydration: 43678621
   TzAutoupdateStandardUser: 49229351
   U3RHDC: 0
@@ -4208,7 +4098,6 @@
   UCPD_TASKBAR: 47443885
   UCPD_TASKBAR_REVISED: 57398953
   UCPD_WSB: 48789334
-  UCSIv2_DriverGlobalsVerifierFix: 0
   UCX_Etw: 0
   UDERI: 0
   UDFDOD: 0
@@ -4291,7 +4180,6 @@
   UsePlainCPPInsteadofCPPWinRT: 56249406
   UseRefactoredDictation: 11252401
   UseSpeechPlatformSettings: 16976317
-  UseStateEndpoint: 37237894
   UseTPMForEnrollmentKey: 31513978
   UseUILanguageForVES: 19372390
   UseUplevelSSU: 40801776
@@ -4322,7 +4210,7 @@
   VAHPR: 42518543
   VAIASHS: 42794809
   VAICSE: 49566288
-  VAIRIO: 58477524
+  VAIRIO: 60466682
   VALTMC: 55362165
   VAMCISS: 57113332
   VANEOSC: 56870548
@@ -4345,7 +4233,6 @@
   VA_LEIRCO: 56761493
   VA_LEOIII: 56632490
   VDSkipFullRes: 33812524
-  VLLRTest: 37169215
   VPAI: 45196154
   VPN_BugFixes_25A: 56321339
   VPN_BugFixes_25B: 57623979
@@ -4363,9 +4250,9 @@
   VPN_BugFixes_25C_VPN_LowPowerCleanup: 59125486
   VRegistry_DisableJScript9Legacy: 55139490
   VT: 57237670
+  VTATFF: 33446203
   VTSFDF: 55392126
   VTpmAttestation: 31368561
-  VaBackedVsm: 43030354
   VaUnsupportedLanguageExperience: 36863320
   ValidateTest: 55798745
   ValueBannerCDMLiteV2: 30785790
@@ -4389,12 +4276,8 @@
   VelTest_ControlledByProxy_EBD_External: 54846949
   VelTest_DD_OnReload: 19195502
   VelocityEligibilityUsingFlightingApi: 8776276
-  VelocityProxyContainerSupport: 51835698
-  VelocityProxyContainerSupport_SetOrResetLayerBootConfigurations: 53386125
   Veltest_OnReload_Dep_DD: 19195496
-  VirtualPrinterFrameworkUpdates: 49199085
   VirtualTouchpadInitFailure: 59062108
-  VirtualTouchpadTipTests: 57578667
   VirtualTouchpadTipTests2508: 58464472
   VoiceAccessClippingFix: 38087802
   VoiceAssistantPositioner: 20484887
@@ -4403,7 +4286,6 @@
   VolumeSnapshotCallerEventLogging: 58336672
   VwSotUpdates: 34146562
   VwUpdates: 27923788
-  W32PTP: 41668769
   W365Boot_BlackScreen_Bugfixes: 46232528
   W365Boot_BluetoothSupportForB2CAndSwitch_BugFixes: 47376559
   W365Boot_Branding: 45493830
@@ -4413,6 +4295,7 @@
   W365Boot_Engineering_Improvements: 46179275
   W365Boot_FailFast_V2_Network: 45896634
   W365Boot_FailFast_V2_Provider: 45897081
+  W365Boot_ImprovedLockScreenImageHandling: 59829915
   W365Boot_ManagedPCSettings: 43529319
   W365Boot_ManagedPCSettings_Account: 46842903
   W365Boot_ManagedPCSettings_Account_Rejuv: 53175704
@@ -4509,6 +4392,7 @@
   WUClientSupportsOfflineScanCabOverrides: 55556486
   WUSAMultiMSURebase: 54267662
   WUSARebaseMultiMSU: 54267669
+  WV2HelpersOneSettingsPayloadResilience: 60552797
   WV2HelpersPlatformIfAvailable: 58273852
   WV2_SWV2M: 57297274
   WVD_MultiSession_User_Configuration: 23685071
@@ -4618,8 +4502,6 @@
   XDRVBC: 59339557
   XHSWUI: 29425493
   XIIS_Test: 34919815
-  XTests: 31692694
-  XXTests: 28847533
   XamlHostedDialogGeneric: 35985309
   XamlHostedDialogHyperlinks: 34037898
   XamlScaling_Tolerance: 58276519
@@ -5019,11 +4901,7 @@
   MSRC49890_20309196: 20309196
   MSRC49893_20309898: 20309898
   MSRC49894_20310019: 20310019
-  MSRC49895_20310073: 20310073
-  MSRC49896_20310121: 20310121
-  MSRC49897_20310146: 20310146
   MSRC49908_20311175: 20311175
-  MSRC49933_20369702: 20369702
   MSRC49960_20370815: 20370815
   MSRC49988_20387640: 20387640
   MSRC50126_20461088: 20461088
@@ -5033,7 +4911,6 @@
   MSRC51569_21326875: 21326875
   MSRC51700_21409766: 21409766
   MSRC51792_21506819: 21506819
-  MSRC52101_21708469: 21708469
   MSRC52192_21842263: 21842263
   MSRC52365_21395503: 21395503
   MSRC52443_22097612: 22097612
@@ -5042,33 +4919,21 @@
   MSRC53249_23061662: 23061662
   MSRC53969_23896785: 23896785
   MSRC54581_23775006: 23775006
-  MSRC54643_23866277: 23866277
   MSRC54733_23928498: 23928498
   MSRC54976_24089711: 24089711
   MSRC55118_24188022: 24188022
   MSRC55531_25276811: 25276811
   MSRC55635_24819047: 24819047
-  MSRC56125_24673962: 24673962
   MSRC56147_24705114: 24705114
-  MSRC56582_25185823: 25185823
-  MSRC56583_25185841: 25185841
   MSRC57129_25520433: 25520433
   MSRC57333_25652608: 25652608
-  MSRC57429_25692297: 25692297
-  MSRC57498_25723431: 25723431
   MSRC57503_25726631: 25726631
   MSRC57524_25743201: 25743201
   MSRC57568_25783926: 25783926
-  MSRC57712_27682885: 27682885
   MSRC57795_25894289: 25894289
-  MSRC57892_25937517: 25937517
   MSRC57968_26505869: 26505869
-  MSRC58295_26172760: 26172760
-  MSRC58296_26173880: 26173880
   MSRC58999_26599509: 26599509
   MSRC59101_26702985: 26702985
-  MSRC59279_26844027: 26844027
-  MSRC59313_26884496: 26884496
   MSRC59936_31011409: 31011409
   MSRC59963_27690723: 27690723
   MSRC60518_28390170: 28390170
@@ -5077,7 +4942,6 @@
   MSRC63361_32291786: 32291786
   MSRC63368_32291794: 32291794
   MSRC63672_31648931: 31648931
-  MSRC63783_31710090: 31710090
   MSRC63868_31808102: 31808102
   MSRC64205_32072023: 32072023
   MSRC64273_32106102: 32106102
@@ -5091,7 +4955,6 @@
   MSRC66727_35211842: 35211842
   MSRC67044_35577074: 35577074
   MSRC67337_35911235: 35911235
-  MSRC67582_36327760: 36327760
   MSRC67719_36475946: 36475946
   MSRC67934_36667985: 36667985
   MSRC67991_36691266: 36691266
@@ -5104,8 +4967,6 @@
   MSRC69283_37522153: 37522153
   MSRC69467_37633910: 37633910
   MSRC69497_37647972: 37647972
-  MSRC69668_37781082: 37781082
-  MSRC69672_37782164: 37782164
   MSRC69864_39557780: 39557780
   MSRC69920_37971314: 37971314
   MSRC70067_38076752: 38076752
@@ -5114,18 +4975,14 @@
   MSRC70353_38224898: 38224898
   MSRC70355_38873226: 38873226
   MSRC70513_38309934: 38309934
-  MSRC70554_39260061: 39260061
   MSRC70863_38553898: 38553898
   MSRC71203_38887517: 38887517
   MSRC71207_38852441: 38852441
-  MSRC71324_39721754: 39721754
-  MSRC71572_39277107: 39277107
   MSRC71579_39279649: 39279649
   MSRC71679_39334242: 39334242
   MSRC71897_39543306: 39543306
   MSRC71903_39551627: 39551627
   MSRC71981_39633715: 39633715
-  MSRC72016_39663169: 39663169
   MSRC72021_39665449: 39665449
   MSRC72030_39694119: 39694119
   MSRC72055_39712738: 39712738
@@ -5134,8 +4991,6 @@
   MSRC72114_39763948: 39763948
   MSRC72115_39764682: 39764682
   MSRC72172_40446102: 40446102
-  MSRC72465_39928434: 39928434
-  MSRC72482_39943298: 39943298
   MSRC72497_39960399: 39960399
   MSRC72546_40168272: 40168272
   MSRC72843_40260557: 40260557
@@ -5175,10 +5030,8 @@
   MSRC77032_44518452: 44518452
   MSRC77081_45078045: 45078045
   MSRC77119_43039773: 43039773
-  MSRC77171_43072781: 43072781
   MSRC77245_44478048: 44478048
   MSRC77391_45078064: 45078064
-  MSRC77470_44522460: 44522460
   MSRC77475_43276718: 43276718
   MSRC77554_43345026: 43345026
   MSRC77562_43348160: 43348160
@@ -5187,7 +5040,6 @@
   MSRC77697_43410998: 43410998
   MSRC77699_43412113: 43412113
   MSRC77754_43455206: 43455206
-  MSRC77818_44358071: 44358071
   MSRC77913_43533648: 43533648
   MSRC77918_43535938: 43535938
   MSRC77990_43602805: 43602805
@@ -5199,28 +5051,13 @@
   MSRC78433_43855146: 43855146
   MSRC78513_43950061: 43950061
   MSRC78529_43897985: 43897985
-  MSRC78560_43910792: 43910792
   MSRC78593_43937850: 43937850
   MSRC78597_43939377: 43939377
-  MSRC78672_43978817: 43978817
   MSRC78843_44061188: 44061188
-  MSRC78917_44116784: 44116784
   MSRC78969_45019308: 45019308
-  MSRC79017_44171633: 44171633
-  MSRC79053_45077756: 45077756
-  MSRC79065_44215680: 44215680
-  MSRC79068_44215792: 44215792
-  MSRC79084_44218509: 44218509
-  MSRC79085_45020020: 45020020
-  MSRC79109_45078424: 45078424
-  MSRC79185_44256906: 44256906
-  MSRC79209_44270724: 44270724
   MSRC79282_44324356: 44324356
-  MSRC79444_45078740: 45078740
   MSRC79554_44585456: 44585456
   MSRC79577_45034938: 45034938
-  MSRC79607_45078776: 45078776
-  MSRC79619_44627059: 44627059
   MSRC79679_45078816: 45078816
   MSRC79680_45078818: 45078818
   MSRC79686_45077436: 45077436
@@ -5232,37 +5069,26 @@
   MSRC80056_45078931: 45078931
   MSRC80067_44857076: 44857076
   MSRC80111_45078894: 45078894
-  MSRC80192_45078901: 45078901
   MSRC80204_45078951: 45078951
-  MSRC80225_45078934: 45078934
   MSRC80460_45140204: 45140204
-  MSRC80513_45190537: 45190537
-  MSRC80577_45248339: 45248339
   MSRC80590_45262569: 45262569
   MSRC80693_45361143: 45361143
   MSRC80700_45361568: 45361568
-  MSRC80787_45413023: 45413023
   MSRC80799_45416767: 45416767
   MSRC80821_45433834: 45433834
   MSRC80905_45522986: 45522986
   MSRC80956_45575697: 45575697
   MSRC80988_45582712: 45582712
-  MSRC81036_45614508: 45614508
   MSRC81058_45628639: 45628639
   MSRC81059_45628743: 45628743
   MSRC81067_45630671: 45630671
   MSRC81095_47057119: 47057119
   MSRC81178_45692242: 45692242
   MSRC81236_45739699: 45739699
-  MSRC81357_45824606: 45824606
-  MSRC81368_45828682: 45828682
   MSRC81398_45839776: 45839776
   MSRC81432_45868315: 45868315
-  MSRC81441_45871520: 45871520
   MSRC81490_45899296: 45899296
-  MSRC81503_45904802: 45904802
   MSRC81525_45929489: 45929489
-  MSRC81533_45934424: 45934424
   MSRC81565_45951547: 45951547
   MSRC81643_45997224: 45997224
   MSRC81675_46023653: 46023653
@@ -5270,52 +5096,34 @@
   MSRC81708_46045627: 46045627
   MSRC81713_46046947: 46046947
   MSRC81769_46091435: 46091435
-  MSRC81795_46109107: 46109107
-  MSRC81841_46189820: 46189820
   MSRC81890_46238158: 46238158
   MSRC81899_46251106: 46251106
   MSRC81952_46340642: 46340642
   MSRC81952_46512934: 46512934
   MSRC81972_46356507: 46356507
-  MSRC81976_46356832: 46356832
   MSRC82022_46378877: 46378877
-  MSRC82040_46416463: 46416463
   MSRC82046_46418054: 46418054
   MSRC82065_46420297: 46420297
   MSRC82074_46421564: 46421564
-  MSRC82219_46499076: 46499076
   MSRC82223_46499850: 46499850
   MSRC82244_46512428: 46512428
-  MSRC82249_46512900: 46512900
   MSRC82283_46530124: 46530124
-  MSRC82342_46550638: 46550638
   MSRC82360_46566943: 46566943
   MSRC82361_46567049: 46567049
   MSRC82365_46569809: 46569809
-  MSRC82368_46570136: 46570136
   MSRC82448_46631299: 46631299
-  MSRC82449_46631415: 46631415
   MSRC82452_46633319: 46633319
   MSRC82566_46731351: 46731351
   MSRC82592_46737846: 46737846
-  MSRC82642_46803032: 46803032
-  MSRC82707_46836114: 46836114
-  MSRC82737_46851271: 46851271
   MSRC82742_46854635: 46854635
   MSRC82786_46894531: 46894531
-  MSRC82819_46938771: 46938771
   MSRC82825_46939803: 46939803
   MSRC82850_46956702: 46956702
-  MSRC82930_47050820: 47050820
   MSRC82963_47088154: 47088154
   MSRC83037_47109561: 47109561
-  MSRC83073_47127159: 47127159
-  MSRC83084_47130607: 47130607
   MSRC83086_47130937: 47130937
   MSRC83098_47146258: 47146258
   MSRC83188_47225054: 47225054
-  MSRC83195_47225807: 47225807
-  MSRC83245_47245019: 47245019
   MSRC83269_47261226: 47261226
   MSRC83271_47261424: 47261424
   MSRC83272_47261494: 47261494
@@ -5328,45 +5136,27 @@
   MSRC83294_47271471: 47271471
   MSRC83323_47321244: 47321244
   MSRC83336_47327431: 47327431
-  MSRC83341_47332581: 47332581
   MSRC83342_47332914: 47332914
   MSRC83349_47343038: 47343038
-  MSRC83386_47380699: 47380699
   MSRC83401_47386370: 47386370
-  MSRC83417_47405491: 47405491
   MSRC83451_47451131: 47451131
-  MSRC83511_47495206: 47495206
-  MSRC83556_47527701: 47527701
   MSRC83596_47607157: 47607157
-  MSRC83622_47634844: 47634844
   MSRC83623_47634948: 47634948
-  MSRC83626_47635637: 47635637
-  MSRC83654_47663842: 47663842
   MSRC83676_47691012: 47691012
   MSRC83677_47691053: 47691053
-  MSRC83685_48791709: 48791709
   MSRC83716_47773605: 47773605
   MSRC83776_47799907: 47799907
-  MSRC83784_47801514: 47801514
-  MSRC83785_47914157: 47914157
   MSRC83823_48864506: 48864506
-  MSRC83875_47913970: 47913970
-  MSRC83895_47921880: 47921880
   MSRC83962_47973003: 47973003
-  MSRC84010_48014487: 48014487
   MSRC84046_48034275: 48034275
   MSRC84131_48066503: 48066503
   MSRC84149_48082365: 48082365
-  MSRC84228_48129623: 48129623
   MSRC84237_48130389: 48130389
   MSRC84263_48139641: 48139641
-  MSRC84518_48266647: 48266647
   MSRC84587_48308642: 48308642
-  MSRC84693_48372589: 48372589
   MSRC84886_48470785: 48470785
   MSRC84907_48484237: 48484237
   MSRC84956_48522668: 48522668
-  MSRC85140_48642897: 48642897
   MSRC85832_49078942: 49078942
   MTestAbSh1: 41799415
   MTestAbShCTU: 47610223
@@ -5803,15 +5593,13 @@
 
 ## Enabled By Default:
 
-  1232627001: 54853644
-  3707728185: 58745129
+  1370426682: 60765642
   38616832: 38616832
   39449421: 39449421
   39606579: 39606579
   4232613179: 58674337
   42480664: 42480664
   43923537: 43923537
-  46090951: 46090951
   46104793: 46104793
   46354629: 46354629
   46956657: 46956657
@@ -5901,7 +5689,6 @@
   54587088: 54587088
   54620742: 54620742
   54629273: 54629273
-  54644138: 54644138
   54644696: 54644696
   54667871: 54667871
   54702919: 54702919
@@ -5938,7 +5725,6 @@
   55481656: 55481656
   55482158: 55482158
   55535480: 55535480
-  55536872: 55536872
   55552674: 55552674
   55652650: 55652650
   55683212: 55683212
@@ -6172,6 +5958,7 @@
   60604361: 60604361
   6230: 60096230
   638880057: 58645271
+  901451066: 60764371
   AADClientSessionKeyRolloverV2: 39844436
   AADEarlyDownloader: 51875814
   AB266: 55753266
@@ -6188,6 +5975,7 @@
   AVCEnc_CodecPack: 48422028
   AadAssets: 55937916
   AadAssetsRestore: 56259285
+  AboutPage_UI_fixes: 59921889
   AccessEnableVSyncEventAtomically: 54532698
   AccessibilityFixKeyboardAndFocus: 57561007
   AccountAndRegionalPolicyEnforcement: 47557358
@@ -6200,19 +5988,19 @@
   AddPreRegisterInfoToRegistryAndReportFromAddDependencyToProcessPackageGraph: 57451871
   AddQrCodeAccessibilityText: 57170623
   AddingCloudFileCheck: 57778831
+  AdditionalClocksSettings_BugFix: 58682398
   AdditionalSVRTracing: 57341397
   AdvancedShaderDelivery: 58442645
   AdvancedSystemSettingsLongPath: 59008083
   AdvancedSystemSettingsPage: 59009713
   AdvancedSystemSettingsVirtualWorkspaces: 59008064
+  AdvancedSystemSettings_UpdateStubApp: 60564288
   AllAppsDeploymentRequestTip: 59316565
   AllowBackupEntraHybrid: 52065424
-  AllowDismProvisionWithExternalLocation: 58338535
   AllowHAMServicingInvalidStateTransition: 58037746
   AllowListForBlockNonAdminUserInstall: 54689274
   AlwaysRepairSystemMetadataTLACEOnRegister: 50393476
   AlwaysResetBackTrackSourceMode: 58492978
-  ApMonImpersonation_Fix: 59112626
   AppExAliasAbsPath2: 53813530
   AppReadinessBaseTaskRace: 59142795
   AppSiloActivationOverride: 50292181
@@ -6226,6 +6014,7 @@
   ApplicationDataApiPerf: 59783591
   AppxManifestReader_RuntimeIsAppSiloButNoTrustLevel: 58368637
   AppxPackaging_Downlevel: 57623388
+  ArmHaadjTaskStateFix: 60488390
   ArsNullCheck: 57902772
   AutoRotation_PreserveContentResolutionOnPrimaryClonePath: 57374666
   AutomaticGamepadInvocation: 56240668
@@ -6238,6 +6027,7 @@
   BTDKS_55832605: 55832605
   BackdropMaxTextureSize: 33019496
   BackupRestoreLogImprovements: 57850079
+  Backup_Key_Rotation: 58974045
   BasicDisplayDeviceEscape_Fix: 57513490
   BestEffortLoadTestHookForStateDEH: 55727128
   BestEffortRemoveExtensionProgIds: 57267967
@@ -6250,10 +6040,10 @@
   BrightnessSettingsTreatAIOAsInternal: 55861683
   BugFix53931218: 53931218
   BugFix56236958: 56236958
+  BugFixHandleErrorGetText: 59583442
   BugFixTextInputHostRootRemoval: 55706882
   BugFix_58608224: 58608224
   BugFix_59275501: 59275501
-  Bugfix_19900353_21622062: 19900353
   Bugfix_43770709: 43770709
   Bugfix_45354447: 45354447
   Bugfix_55037899: 55037899
@@ -6307,7 +6097,6 @@
   CaptureWaitChainDumpsOnHang: 37717200
   Capture_AutoParentedFilteredCaptureFix: 50776729
   Capture_BufferCollection: 51855954
-  Capture_CAMIGraphicsCaptureItemInteropScoped: 57453622
   Capture_COMApartmentFix: 59389569
   Capture_CaptureManagerBackport: 60445156
   Capture_DirtyRegionOcclusion: 59015030
@@ -6320,11 +6109,11 @@
   Capture_PartiallyInitializedDisplayEntry: 49013427
   Capture_uDWMFixMultipleWindowCaptures: 57085940
   CentennialActivationUsesShellExecuteNoHooksMask: 60090173
-  ChangeJsonParserForDownloadSdoConfig: 54336074
   CheckCanConnectTargetBeforeConnectTarget: 52121215
   CheckPopupHwndIsWindow: 31252183
   CheckUndockedModulePackagePresent: 51953184
   CimFix: 51868790
+  CleanTargetScalingIntentCache: 57514170
   CleanUpDisplayedPrimaryInSetTimings: 58537985
   CleanupOutdatedPackagesCausedByDynamicDependencyInUse: 56897923
   ClickSensitivity: 32720331
@@ -6376,7 +6165,6 @@
   D3D999E5Revision: 45279028
   DCP_Edge_Enlightenment: 53001506
   DDisplayInkBlackFlash: 59893861
-  DDisplayPreserveVidpn: 59947852
   DDisplayScanoutCacheFlags: 57084425
   DDisplay_AggCheckMpo: 53851848
   DDisplay_HidePrivateFormats: 56865324
@@ -6399,7 +6187,6 @@
   DelegationInCoreInputView_58789108: 58789108
   DeleteCachedInboxManifestWhenFolderMoveDetected: 59219482
   DeploymentNotificationsOptimization2: 44175363
-  DeviceAssociation_ge_10d_bugfixes: 50494401
   DeviceInterfaceQuery_Fix: 59701372
   DeviceInventoryAvoidRegionSelectionDeletedPackages: 58407310
   DisableAutoRotationOnGameHandheld: 59442499
@@ -6418,8 +6205,8 @@
   DisplayBugBundle_2507: 57512306
   DisplayBugBundle_2508: 57512307
   DisplayBugBundle_2509: 57512308
-  DisplayBugBundle_2601: 59104222
-  DisplayBugBundle_2602: 60133026
+  DisplayBugBundle_2510: 58650682
+  DisplayBugBundle_2511: 58650683
   DisplayChangeDismiss: 56863738
   DisplayDitheringSupport: 34861659
   DisplayMuxDisablePostTestFailureSwitch: 60259634
@@ -6427,6 +6214,7 @@
   DisplayMux_PostGA_BugBundle_2: 57319954
   DisplayPackageFullNameOnAliasPage: 58323029
   DisplaySettingsTargetSyncOnViewInvalidation: 57017005
+  DmaSsoAADCompliance: 60413865
   DodVsyncEnsureLocked: 54538721
   DolbyVisionAndUpdatedHdrUx: 49793372
   DolbyVisionForcePresent: 57247231
@@ -6469,7 +6257,7 @@
   ExtendedUIXamlRefresh: 29532725
   ExternalBrightness_UIErrorReporting: 54920936
   EyeControllerApp_Crash2508: 58286846
-  EyeControllerApp_LaunchTip: 57374882
+  FAQ_Flicker_BugFix: 59979601
   FEHI: 51617480
   FE_PI_Perf_2: 57407887
   FE_SharedBackend: 47135408
@@ -6510,7 +6298,6 @@
   FixDeviceStopRestartForAzureStorageRepaveNode: 59711287
   FixDisplayIDMonitorSizeCalculation: 60247252
   FixDolbyVisionOnNewPath: 51878904
-  FixFlipManagerTokenListCorruptionBugcheck: 57100058
   FixForShutdownWithAgentSession: 59783318
   FixGetApartmentTypeNullAV: 60297995
   FixIdd112Without111: 59252123
@@ -6520,7 +6307,9 @@
   FixModernStandbyDRIPSForMultipleSSD: 56778738
   FixNativeNVMeLEDIndication: 57794307
   FixNonAtomicAccessInCLegacyCatalog: 57632793
+  FixNonAtomicAccessInCTypeInfo2: 60473431
   FixNonAtomicAccessInCachedRegistryKeyData: 57632807
+  FixNonAtomicAccessInCombaseAndRpcss: 60473437
   FixNonAtomicAccessInDComRem: 57527843
   FixNonAtomicAccessInDllCache: 57632806
   FixNonAtomicAccessInInterceptor: 57632794
@@ -6562,7 +6351,6 @@
   GamepadInvokesOnAllowedList: 57708763
   GamepadInvokesOnAllowedList2508: 58305785
   GamepadInvokesOnAllowedList2510: 59193861
-  GamepadVKeyRoutingOptIn: 55740572
   GuestUserLogonWAMSSO: 45710763
   GuestUserRDPToAVD: 45682905
   H264_DRMLowLatency: 52407192
@@ -6698,7 +6486,6 @@
   IppUsbExclusionList_For_Scanners: 58535149
   IsolatedMachinePassword: 35570258
   IssuerSid_AltSecId: 56526528
-  JapaneseTouchKeyboardTypingIssue: 51408638
   KVS_BugFixes_54616183: 54616183
   KVS_BugFixes_54616193: 54616193
   KVS_BugFixes_55830739: 55830739
@@ -6756,6 +6543,7 @@
   MCITF: 57629258
   MJPEG_DXVA_IMP_2409: 49642928
   MPAFCP: 58743488
+  MSPhotographyUnloadFix: 57490427
   MSRC78227_43813644: 43813644
   MSRC79199_45078457: 45078457
   MSRC79610_45078752: 45078752
@@ -6830,12 +6618,14 @@
   MediaQI2505: 55203471
   MediaQI2508: 56952301
   MediaQI2511: 58156969
+  MediaQI2602: 59423145
   MercuryADEPTAppBackup: 41984359
   MercuryADEPTDeviceMetaDataBackup: 43469028
   MercuryADEPTUserIntentBackup: 44140851
   MercuryAppBackup_Post2309: 45949950
   MercurySettingsStateCapture: 46619189
   ModalDialogBroker: 55444028
+  MostRecentlyStagedByPackageId: 57438569
   MoveRegionValidationToSettingsEnvironment: 59266548
   MsalJsWamBridge: 35412656
   MsfteditOptimizationAVFix: 54462134
@@ -6873,6 +6663,7 @@
   OSSwapLOBAppFix: 59214541
   OS_Swap_Preregister_PlayB: 58746726
   OTRInvestigationImprovements: 59193252
+  OfflineFilesPerClassIcon: 60610656
   OmitSetEventOnNullHandleInDeploymentRequest: 54842180
   OptFeatSettHideTextWhenNA: 53522474
   OptimizationsInApplistBackup: 46406346
@@ -6923,7 +6714,6 @@
   Print_TCPMon_PortDeadlock_Fix: 58601332
   Promote_Unpreferred_Escl_Scanner: 59163113
   ProvisionAndPinnedUupPackageResolution: 58961007
-  ProvisionedExternalLocationInAllUserStore: 59976253
   RACFL: 58230865
   RCCBTest: 54278070
   RWAMD: 58546654
@@ -6941,7 +6731,6 @@
   RepairShouldReturnError: 56211925
   ReportHangAPI: 54722784
   RequeueClearVariablesAndRepositorySession: 58757712
-  RequireOnlyMachineExternalLocationOnProvisionedAppsUsingExternalLocation: 56298040
   RestartDshowBridgeOnCrash: 48430399
   RestoreCSPIntuneReporting: 58073768
   ReturnFailureIfHAMCallFails: 57200791
@@ -6951,8 +6740,10 @@
   SALT: 57691547
   SBFTest: 51942247
   SCF: 53046817
+  SCFTest: 45425284
   SCHDFF: 58453249
   SCIBF7dTest: 51359559
+  SDIRL: 60570513
   SDOBadParameterBugFix: 59681077
   SFI01075: 58101075
   SFI177: 58101177
@@ -6968,7 +6759,6 @@
   SFS_CD_BugFixes_2511: 58550741
   SGAndInterleavedH264WithMepOptIn: 59175713
   SHTC: 49343691
-  SIP53391992: 53391992
   SISBOTest: 53654840
   SIT: 57590744
   SMB1: 55920864
@@ -6981,6 +6771,7 @@
   SWT_2: 55006994
   SWT_3: 56859802
   SWT_DP: 59229517
+  SafeDodBltQueueAccesses: 57632977
   SavePairedAdapterInRemoteSession: 58315719
   ScalarMultiplierForBrightness: 55876058
   Scenario55427464: 55427464
@@ -6996,6 +6787,7 @@
   SensorsBugfixBundle2508: 57605810
   SeparateBRFromESR: 52060425
   Servicing_AdaptiveXboxSeek: 57952288
+  Servicing_AddSupportForPureMlDsa: 59058596
   Servicing_AutoRotation_UseWeakReferenceOnFactory: 56400221
   Servicing_BLEHeapLeakFix: 52049456
   Servicing_CertEnroll_SynchronizeResourceStringsCache: 38859679
@@ -7006,13 +6798,11 @@
   Servicing_KcmModalWindowSupport: 58582361
   Servicing_MFDynamicArrayOverflowCheck: 59119875
   Servicing_MSRC63779: 56457100
-  Servicing_MSRC65292: 56456836
   Servicing_NullCheckTimedTextMediaSource: 51858037
   Servicing_PlaybackItem_Resolved_Race_Condition: 60336434
   Servicing_Prefer_HEVC_OEM_Over_HEVC_Consumer: 57398416
   Servicing_RoundedImageSupport: 49218143
   Servicing_ShellDataModel_HiddenPidlUnderflow: 55990087
-  Servicing_TSF_RDPIME_Reconnect: 55998560
   Servicing_TaskMgrInitAV_39201110: 39201110
   Servicing_ThumbnailForVideo422And444Format: 58142550
   Servicing_TimedTextNoEvtHandlerException: 51265269
@@ -7053,7 +6843,6 @@
   SmsRouter_Bugfixes: 53856990
   SpatialAudioMultipleAppServices: 56500497
   SpectreUsageTelemetry: 58287508
-  SpeechRuntime_1_46_0_LiveCaptions: 59337145
   Spooler_RPC_Authentication_Improvements: 58777735
   StageRequeuesOnRegisterHangWithUUPFix: 59897203
   StagingCopyDuplicateFiles: 59721958
@@ -7099,14 +6888,12 @@
   TipTestsForPreRegisterInboxPackageScenario: 57278717
   TopCardUIUpdates: 57501878
   TrackOtrRegistrationStatusInPackageFamilyList: 58688068
-  TraditionalMongolianKeyboardLayout: 49081368
   TrustedLaunch: 55049681
   TrustedLaunchHosts: 58019456
   TrustedLaunchSysApps: 55049707
   TrustedLaunchWithDevModeAndTestSigning: 59273986
   TrustedLaunch_IgnoreForF5: 59289735
   TryShowHidesImmediatelyV2: 59043595
-  TulipAccessibilityBugFixes2: 55675385
   UAC_CPL_57068203: 57068203
   UCAIF: 58754686
   UCCDPA: 58881058
@@ -7123,6 +6910,7 @@
   UpdateCryptoAPIs: 57095419
   UpdateTaskBugfix: 46009404
   UseAEAForUnsupportedPackagedCreateProcessScenario: 57441842
+  UseCentralWindowsAppRuntimeVersion: 58375066
   UseInitOnceAndSRCacheForMuaCheck: 33965401
   UseReadPointerNoFenceInIsOwner: 57998985
   UserBindingForEnclaves: 58180965
@@ -7173,6 +6961,7 @@
   WorkFolderSync: 43739851
   WwanApi_HandleTable_Cleanup_BUGFIX: 58090271
   XVP_AVD444Fix: 52608522
+  XamlRootCrashFix59885015: 59885015
   YUVSwapchainShaderInputUsageCD: 58937907
 
 ## Disabled By Default:
@@ -7270,12 +7059,14 @@
   AV1Enc_CodecPack_Enumerate_First: 48435294
   AVCEnc_CodecPack_Enumerate_First: 45598712
   AboutPageUIEnhancements: 59265307
-  AboutPage_UI_fixes: 59921889
   AcknowledgeReceiptOnLoad: 43200402
   AddRegkeyForceUseWebview1: 59998409
   AdminlessConsumerConfig: 51406324
+  AdvancedSystemSettingsWSL: 58182453
+  AllowDismProvisionWithExternalLocation: 58338535
   AllowImpersonationTokenToWorkOnCallToCdsUdkApis: 41309836
   AlwaysOpenFoldersInNewTab: 49143212
+  ApMonImpersonation_Fix: 59112626
   AppExAliasAbsPath: 51578581
   AppSiloFileBrokerCoreWindowCheck: 55053776
   AppXLaunchPerformanceCache: 58146454
@@ -7286,12 +7077,12 @@
   AutoIdentityGenerationHook: 55676379
   AvoidLoggingErrorForGetPartition: 58793642
   BCFMAATA: 60086113
-  Backup_Key_Rotation: 58974045
+  BFDTSHH_Test: 60597084
+  BackupStatusFromBRC: 44973308
   BlockDMFT_LVFDMFT_2511: 54414708
   BlockDMFT_RsDMFT_2511: 59194296
   BugFix52931714: 52931714
   BugFix56414827: 56414827
-  BugFixHandleErrorGetText: 59583442
   BugUserProfilePath: 38839574
   Bugfix53937309: 53937309
   Bugfix_57240160: 57240160
@@ -7305,6 +7096,7 @@
   CAgileReferenceMarshaledDestructorReleaseMarshalObjRefInCurrentApartment: 58286439
   CBFF: 58267509
   CBSIF: 60153526
+  CCFIMAAWTA: 60626390
   CD_MJPEG_MFT_Decoder: 46913462
   CDependencyPropertyUASFix: 58755789
   CESF: 58739770
@@ -7336,6 +7128,7 @@
   CompSwapchainRenderAndPresentSync: 53994671
   Crumbs: 40950262
   D3D11ShaderAccessRestrictedResource: 50036719
+  DDisplayPreserveVidpn: 59947852
   DICFO: 60387398
   DPAPI_GUI_Telemetry: 57702893
   DPLWI: 59732831
@@ -7354,8 +7147,8 @@
   DisableDXVA_AV1: 36499710
   DisableDXVA_HEVC: 36500243
   DisableDXVA_VP9: 36500244
-  DisplayBugBundle_2510: 58650682
-  DisplayBugBundle_2511: 58650683
+  DisplayBugBundle_2601: 59104222
+  DisplayBugBundle_2602: 60133026
   DisplayMux_ForceDriverFullSupport: 48160154
   DisplayMux_ForceMuxFullSupport: 54920706
   ECU8397: 53788397
@@ -7375,7 +7168,6 @@
   ExperimentalCompositorApi: 57056490
   ExposeRuntimePriorityDeploymentOptionsInPublicAPIs: 56794222
   ExternalBrightness: 12759424
-  FAQ_Flicker_BugFix: 59979601
   FDPAD: 55608277
   FE_LPC: 51848093
   FE_PI: 48895801
@@ -7399,6 +7191,7 @@
   FrameServer_WorkqueueFix: 58101251
   GPRoamingControl: 57385861
   GetHelp_SettingsSupportLinks: 57606594
+  GetPreferredThreadLanguage_Watson_AccessViolation_BugFix: 59362864
   HEVCEnc_CodecPack: 48435131
   HEVCEnc_CodecPack_Enumerate_First: 48435339
   HWDRMByDefault: 56947982
@@ -7427,6 +7220,7 @@
   ITMWF: 48187829
   IVCFTest: 57793144
   IgnoreLoggingTimeoutErrors: 60093228
+  IgnoreUserProfileUnavailableError: 59849777
   ImpB_WH_59294544: 59294544
   InputHostErrorHandlingFix: 52486497
   InvalidCastPrevention: 48038103
@@ -7445,8 +7239,6 @@
   MatchTagComputationForTelemetry: 59813225
   MbbCx_Fix_OutOfScopeReadsInResponseHandlers: 60211987
   Mbbcx_Fix_Ip_Configuration_After_Disconnect: 54656897
-  MediaQI2602: 59423145
-  MostRecentlyStagedByPackageId: 57438569
   NDU_Mitigate_InterfaceStatsAccessViolation: 59451597
   NLM_DeadlockMonitor: 43203845
   NLM_Fix_FilterHiddenNetworkInterface: 59811951
@@ -7479,6 +7271,7 @@
   PresenceSettingsViaNavigateTo: 58935493
   Print_PlatformStabilizationFixes_2026_Wave1: 57741219
   Print_PrinterRename_Fix: 58773484
+  ProvisionedExternalLocationInAllUserStore: 59976253
   Psa_V5_Api: 58053430
   Psa_V5_Api_Implementation: 58053437
   RC4Deprecation: 52511895
@@ -7488,11 +7281,11 @@
   RemoveDefaultMicrosoftStorePackages_EnableOnAllOSVersions: 58250290
   RemoveDefaultMicrosoftStorePackages_MultiSession: 57056159
   RepairIfMissingExtensions: 52973182
+  RequireOnlyMachineExternalLocationOnProvisionedAppsUsingExternalLocation: 56298040
   SASBF: 53446662
   SATCS: 57146208
   SCFF3PTest: 52441334
   SCFFATest: 50658022
-  SCFTest: 45425284
   SEHS_FTA: 59337016
   SEWF: 54449542
   SFS_CD_BugFixes_2601: 59818596
@@ -7505,8 +7298,9 @@
   STP_CAPS: 59684742
   SWATCS: 57378454
   SWT_4: 58527096
+  SWT_CF: 60686121
   SWT_PL: 60362048
-  SafeDodBltQueueAccesses: 57632977
+  ScanAllAltSecIds: 59794829
   SearchIndexerMultiSessionLockFix: 46500419
   SecureDownload: 58274776
   SemanticSearch_Improved_Diagnostics: 57616232
@@ -7517,11 +7311,10 @@
   Servicing_56775829: 56775829
   Servicing_StartMenuFixJapaneseSorting: 44143798
   SetMonitorDescriptorOverride: 55797135
+  Settings_ImpB_WH_60547142: 60547142
   ShadowAdmin: 59275076
   ShareExperiment: 58047213
   SmartScreenGroupPolicyAndUriReputationCheck: 48157573
-  SpeechRuntime_1_46_0_Narrator: 59336768
-  SpeechRuntime_1_46_0_VoiceAccess: 59337133
   SpeechRuntime_1_46_0_VoiceTyping: 59379618
   StageRequeuesOnRegisterHang: 59361280
   StateRepository_TileView_GetMaxChangeIdByUser_QueryOptimization: 56194618
@@ -7566,11 +7359,13 @@
   WCM_Fix_ForceDisconnectInStandbyForSpecificHardware: 57690839
   WFRPCSO: 59827312
   WHA: 58193426
+  WTA: 60538031
   WWAN_Fix_DispatchNotification_Illegal_CWwanManagerSelectInterface_Exception: 57112000
   WdfUmdfDma: 44314290
   WdfUmdfOrderedIoCompletion: 57783350
   WebAccountUPNFix: 50937614
   WinUsb_Print: 56737116
+  WindowsAppRuntime_NextIsDefault: 58375086
   WindowsHelloAdoptionEnS: 58433043
   WindowsHelloForLockLogonUIWithLottie: 57176675
   WindowsProtectedPrintSpoolerWorker: 58777675
